<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="David N. Nicholson" />
  <meta name="author" content="Vincent Rubinetti" />
  <meta name="author" content="Dongbo Hu" />
  <meta name="author" content="Marvin Thielk" />
  <meta name="author" content="Lawrence E. Hunter" />
  <meta name="author" content="Casey S. Greene" />
  <meta name="dcterms.date" content="2020-11-18" />
  <meta name="keywords" content="biorxiv, preprints, pubmed-central, natural-language-processing, descriptive-linguistics" />
  <title>Linguistic Analysis of the bioRxiv Preprint Landscape</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/master/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="Linguistic Analysis of the bioRxiv Preprint Landscape" />
  <meta name="citation_title" content="Linguistic Analysis of the bioRxiv Preprint Landscape" />
  <meta property="og:title" content="Linguistic Analysis of the bioRxiv Preprint Landscape" />
  <meta property="twitter:title" content="Linguistic Analysis of the bioRxiv Preprint Landscape" />
  <meta name="dc.date" content="2020-11-18" />
  <meta name="citation_publication_date" content="2020-11-18" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="David N. Nicholson" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania" />
  <meta name="citation_author_orcid" content="0000-0003-0002-5761" />
  <meta name="twitter:creator" content="@None" />
  <meta name="citation_author" content="Vincent Rubinetti" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania" />
  <meta name="citation_author_orcid" content="None" />
  <meta name="twitter:creator" content="@vincerubinetti" />
  <meta name="citation_author" content="Dongbo Hu" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania" />
  <meta name="citation_author_orcid" content="None" />
  <meta name="twitter:creator" content="@None" />
  <meta name="citation_author" content="Marvin Thielk" />
  <meta name="citation_author_institution" content="Elsevier" />
  <meta name="citation_author_orcid" content="0000-0002-0751-3664" />
  <meta name="twitter:creator" content="@None" />
  <meta name="citation_author" content="Lawrence E. Hunter" />
  <meta name="citation_author_institution" content="Center for Computational Pharmacology, University of Colorado Denver School of Medicine, Aurora, CO, USA" />
  <meta name="citation_author_orcid" content="0000-0003-1455-3370" />
  <meta name="twitter:creator" content="@None" />
  <meta name="citation_author" content="Casey S. Greene" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania" />
  <meta name="citation_author_orcid" content="0000-0001-8713-9213" />
  <meta name="twitter:creator" content="@None" />
  <link rel="canonical" href="https://greenelab.github.io/annorxiver_manuscript/" />
  <meta property="og:url" content="https://greenelab.github.io/annorxiver_manuscript/" />
  <meta property="twitter:url" content="https://greenelab.github.io/annorxiver_manuscript/" />
  <meta name="citation_fulltext_html_url" content="https://greenelab.github.io/annorxiver_manuscript/" />
  <meta name="citation_pdf_url" content="https://greenelab.github.io/annorxiver_manuscript/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://greenelab.github.io/annorxiver_manuscript/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://greenelab.github.io/annorxiver_manuscript/v/cc95cbc39a433e6a2a6993190ca7eff86f6858a7/" />
  <meta name="manubot_html_url_versioned" content="https://greenelab.github.io/annorxiver_manuscript/v/cc95cbc39a433e6a2a6993190ca7eff86f6858a7/" />
  <meta name="manubot_pdf_url_versioned" content="https://greenelab.github.io/annorxiver_manuscript/v/cc95cbc39a433e6a2a6993190ca7eff86f6858a7/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Linguistic Analysis of the bioRxiv Preprint Landscape</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://greenelab.github.io/annorxiver_manuscript/v/cc95cbc39a433e6a2a6993190ca7eff86f6858a7/">permalink</a>)
was automatically generated
from <a href="https://github.com/greenelab/annorxiver_manuscript/tree/cc95cbc39a433e6a2a6993190ca7eff86f6858a7">greenelab/annorxiver_manuscript@cc95cbc</a>
on November 18, 2020.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>David N. Nicholson</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0002-5761">0000-0003-0002-5761</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/danich1">danich1</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania
· Funded by The Gordon and Betty Moore Foundation (GBMF4552); The National Institutes of Health (T32 HG000046)
</small></p></li>
<li><p><strong>Vincent Rubinetti</strong><br>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/vincerubinetti">vincerubinetti</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/vincerubinetti">vincerubinetti</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania
</small></p></li>
<li><p><strong>Dongbo Hu</strong><br>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/dongbohu">dongbohu</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania
</small></p></li>
<li><p><strong>Marvin Thielk</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0751-3664">0000-0002-0751-3664</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/MarvinT">MarvinT</a><br>
<small>
Elsevier
</small></p></li>
<li><p><strong>Lawrence E. Hunter</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1455-3370">0000-0003-1455-3370</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/LEHunter">LEHunter</a><br>
<small>
Center for Computational Pharmacology, University of Colorado Denver School of Medicine, Aurora, CO, USA
</small></p></li>
<li><p><strong>Casey S. Greene</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8713-9213">0000-0001-8713-9213</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/cgreene">cgreene</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania
</small></p></li>
</ul>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<p>Preprints allow researchers to make their results quickly and widely accessible to the scientific community.
These are scholarly works that have yet to undergo the peer review process and are often hosted within open access repositories such as bioRxiv.
The majority of studies involving bioRxiv have focused on preprint metadata; however, a fundamental piece that is missing is an understanding of the language contained within preprints and how it changes through the peer review process.<br />
We sought to compare and contrast linguistic features within bioRxiv preprints to their published counterparts within Pubmed Central’s Open Access corpus (PMC).<br />
We quantified the time delay preprints face while undergoing the peer review process.
We generated document embeddings for every article within bioRxiv and PMC.
We used these embeddings to idenitfy missing preprint-publication links along with training machine learning models to predict journal endpoints for published articles.
We found that topic-specific terms such as “genome”, “neuron” and “network” were enriched within bioRxiv compared to PMC, reflecting bioRxiv’s uptake in bioinformatics and neuroscience preprints.
We found that the leading source of linguistic variation among preprints captured the distinction between quantitative and cellular biology.
We discovered that preprints are delayed an average of 16 days as changes are requested from the peer review process.
Lastly, we created a web app allows anyone to input a bioRxiv or medRxiv preprint and receive a set of the most linguistically similar journals and articles to serve as potential publication venues for their work.</p>
<h2 id="introduction">Introduction</h2>
<p>Preprints are scholarly works that are shared before they have been formally peer reviewed and published.
The practice of sharing preprints has a long history <span class="citation" data-cites="ePmdRbmq">[<a href="#ref-ePmdRbmq" role="doc-biblioref">1</a>]</span>.
The longest ongoing use started with physicists in the 1990s <span class="citation" data-cites="lN9ljhid">[<a href="#ref-lN9ljhid" role="doc-biblioref">2</a>]</span>.
Preprints were used in the life sciences community during the 1960s before publisher pressure stopped the practice <span class="citation" data-cites="ePmdRbmq">[<a href="#ref-ePmdRbmq" role="doc-biblioref">1</a>]</span>.
Over the past decade preprints have made a resurgence within the life sciences community <span class="citation" data-cites="i9BKPuqb QHX7Bezs">[<a href="#ref-i9BKPuqb" role="doc-biblioref">3</a>,<a href="#ref-QHX7Bezs" role="doc-biblioref">4</a>]</span>.
Preprints are now becoming widely accepted and used within the life sciences and other communities <span class="citation" data-cites="PmF48i62 1EwmaUwf9 XqZPlHz7 IyRO6uC Cy352E90 PmF48i62 jXP4z24U YmvKU4HJ">[<a href="#ref-PmF48i62" role="doc-biblioref">5</a>,<a href="#ref-PmF48i62" role="doc-biblioref">5</a>,<a href="#ref-1EwmaUwf9" role="doc-biblioref">6</a>,<a href="#ref-XqZPlHz7" role="doc-biblioref">7</a>,<a href="#ref-IyRO6uC" role="doc-biblioref">8</a>,<a href="#ref-Cy352E90" role="doc-biblioref">9</a>,<a href="#ref-jXP4z24U" role="doc-biblioref">10</a>,<a href="#ref-YmvKU4HJ" role="doc-biblioref">11</a>]</span>.
Common preprint repositories include arXiv <span class="citation" data-cites="ZTyQYlDz">[<a href="#ref-ZTyQYlDz" role="doc-biblioref">12</a>]</span>, bioRxiv <span class="citation" data-cites="i9BKPuqb">[<a href="#ref-i9BKPuqb" role="doc-biblioref">3</a>]</span> and medRxiv <span class="citation" data-cites="NKHNs9j3">[<a href="#ref-NKHNs9j3" role="doc-biblioref">13</a>]</span>; however, there are over 60 different repositories available <span class="citation" data-cites="lMj4YzIP">[<a href="#ref-lMj4YzIP" role="doc-biblioref">14</a>]</span>.</p>
<p>The scientific community has begun to analyze the impact of preprints in the life sciences.
Preprints are being posted at an increasing rate <span class="citation" data-cites="1DCEzEMkZ">[<a href="#ref-1DCEzEMkZ" role="doc-biblioref">15</a>]</span>.
Preprints are also rapidly shared on social media, routinely downloaded, and cited <span class="citation" data-cites="PJwR7lcq">[<a href="#ref-PJwR7lcq" role="doc-biblioref">16</a>]</span>.
Articles with matching preprint versions are cited and discussed more often than articles without them <span class="citation" data-cites="13dwNbYs7 49GeQtua">[<a href="#ref-13dwNbYs7" role="doc-biblioref">17</a>,<a href="#ref-49GeQtua" role="doc-biblioref">18</a>]</span>.
Certain categories of preprints seem to be read and shared differently by both scientists and non-scientists <span class="citation" data-cites="tbWPLbOF">[<a href="#ref-tbWPLbOF" role="doc-biblioref">19</a>]</span>.
Across preprint servers, analyses suggest that between two-thirds to three-quarters of preprints are eventually published <span class="citation" data-cites="QHX7Bezs LcHjf487">[<a href="#ref-QHX7Bezs" role="doc-biblioref">4</a>,<a href="#ref-LcHjf487" role="doc-biblioref">20</a>]</span>.
The time required for a preprint to be published can vary from preprint to preprint; however, preprints with a single version often take less time to publish than preprints with multiple versions <span class="citation" data-cites="tTgEr7g1">[<a href="#ref-tTgEr7g1" role="doc-biblioref">21</a>]</span>, suggesting that authors may update their preprints between submissions for peer review.</p>
<p>Studies of life sciences preprints have primarily focused on the metadata associated with these articles; however, their textual content remains unexamined.</p>
<p>We sought to understand the language landscape of preprints by performing a linguistic analysis of the <em>bioRxiv</em> corpus.
We examined textual differences between preprints and published literature by comparing the entire corpus of preprints with articles available in the open access PubMed Central repository.
We also examined linguistic differences between preprints and their corresponding published pairs.
Examining this shift will provide a unique opportunity to ascertain parts of the peer review and publishing process and how it impacts the scholarly literature.
Neural-network derived document embeddings provide a useful space for determining the textual similarity of preprints, which enables us to extend this work beyond word frequencies.
Examining articles with particularly close proximity in this space reveals unannotated preprint-publication pairs that earlier analyses could not consider.
In this space a preprint’s nearest neighbors are also more likely than distant articles to share an eventual publishing venue with the preprint itself.
We provide a webserver that will displays neighboring journals and articles for any preprint on <em>bioRxiv</em> or <em>medRxiv</em>, which can help authors identify similar papers or suitable journals.
Our linguistic analysis, the first of the bioRxiv corpus, reveals the impact of the life sciences publishing process, introduces a method to identify matching preprint-published article pairs, demonstrates that the text content of preprints is related to their eventual publication venue, and provides a more complete picture of the fraction of preprints that are eventually published.</p>
<h2 id="materials-and-methods">Materials and Methods</h2>
<h3 id="corpora-examined">Corpora Examined</h3>
<h4 id="biorxiv-corpus">BioRxiv Corpus</h4>
<p>BioRxiv <span class="citation" data-cites="i9BKPuqb">[<a href="#ref-i9BKPuqb" role="doc-biblioref">3</a>]</span> is a repository for life sciences preprints.
We downloaded an xml snapshot of this repository on February 3, 2020 from bioRxiv’s Amazon S3 bucket <span class="citation" data-cites="9gVmEi52">[<a href="#ref-9gVmEi52" role="doc-biblioref">22</a>]</span>.
This snapshot contained the full text and image content of 98,023 preprints.
Preprints on bioRxiv are versioned, and in our snapshot 26,905 out of 98,023 contained more than one version.
When preprints had multiple versions, we used the latest one unless otherwise noted.
Authors submitting preprints to <em>bioRxiv</em> select one of twenty-nine different categories.
Researchers also select an article type, which can be a new result, confirmatory finding, or contradictory finding.
Some preprints in this snapshot were withdrawn from bioRxiv: when this happens their content is replaced with the reason for withdrawal.
As there were very few withdrawn preprints, we did not treat these as a special case.</p>
<h4 id="pubmed-central-open-access-corpus">PubMed Central Open Access Corpus</h4>
<p>PubMed Central (PMC) <span class="citation" data-cites="u294RvPz">[<a href="#ref-u294RvPz" role="doc-biblioref">23</a>]</span> is a repository that contains free-to-read articles.
PMC articles can be closed access ones from research funded by the United States National Institutes of Health (NIH) appearing after an embargo period or those published under Gold Open Access <span class="citation" data-cites="iyblQx2M">[<a href="#ref-iyblQx2M" role="doc-biblioref">24</a>]</span> publishing schemes.
Paper availability within PMC is largely dependent on the journal’s participation level <span class="citation" data-cites="lBh7pyRc">[<a href="#ref-lBh7pyRc" role="doc-biblioref">25</a>]</span>.
Individual journals can fully participate in submitting articles to PMC, selectively participate sending only a few papers to PMC, only submit papers according to NIH’s public access policy <span class="citation" data-cites="vGaAJBqg">[<a href="#ref-vGaAJBqg" role="doc-biblioref">26</a>]</span>, or not participate at all.
As of September 2019, PMC had 5,725,819 articles available <span class="citation" data-cites="rzmpIYmn">[<a href="#ref-rzmpIYmn" role="doc-biblioref">27</a>]</span>.
Out of these 5 million articles, about 3 million were open access and available for text processing systems <span class="citation" data-cites="rURwqquJ 18ZyyTcTe">[<a href="#ref-rURwqquJ" role="doc-biblioref">28</a>,<a href="#ref-18ZyyTcTe" role="doc-biblioref">29</a>]</span>.
We downloaded a snapshot of this open access subset on January 31, 2020.
This snapshot contained many types of papers: literature reviews, book reviews, editorials, case reports, research articles and more.
We used only research articles, which aligns with the intended role of <em>bioRxiv</em>, and we refer to these articles as the PMCOA Corpus.</p>
<h4 id="the-new-york-times-annotated-corpus">The New York Times Annotated Corpus</h4>
<p>The New York Times Annotated Corpus (NYTAC) is <span class="citation" data-cites="18JcY2okY">[<a href="#ref-18JcY2okY" role="doc-biblioref">30</a>]</span> is collection of newspaper articles from the New York Times dating from January 1, 1987 to June 19, 2007.
This collection contains over 1.8 million articles where 1.5 million of those articles have undergone manual entity tagged by library scientists <span class="citation" data-cites="18JcY2okY">[<a href="#ref-18JcY2okY" role="doc-biblioref">30</a>]</span>.
We downloaded this collection on August 3rd, 2020 from the Linguistic Data Consortium (see Software and Data Availability section) and used the entire collection for our corpora comparison analysis.</p>
<h3 id="comparing-corpora">Comparing Corpora</h3>
<p>We compared the bioRxiv, PMCOA, and NYTAC corpora to assess the similarities and differences between them.
We use the NYTAC as an out-group to assess the similarity of two life sciences repositories when compared with non-life sciences text.
The corpora contain both words and non-word symbols (e.g., <span class="math inline">\(\pm\)</span>), which we refer to together as tokens to avoid confusion.
We calculated the following statistics for each corpus: the number of documents, the number of sentences, the total number of tokens, the number of stopwords, the average length of a document, the average length of a sentence, the number of negations, the number of coordinating conjunctions, the number of pronouns and the number of past tense verbs.
Next, we used spaCy’s “en_core_web_sm” model <span class="citation" data-cites="q2fFAZTG">[<a href="#ref-q2fFAZTG" role="doc-biblioref">31</a>]</span> (version 2.2.3) to preprocess all corpora and filtered out 326 spaCy-provided stopwords.</p>
<p>Following cleaning, we calculated the frequency of every token across all corpora.
Because many tokens were unique to one set or the other and observed at low frequency, we used the union of the top 100 most frequent tokens from each pair of corpora to compare them.
We generated a contingency table for each token in this union and calculated the odds ratio and 95% confidence interval <span class="citation" data-cites="Lgm5q6mO">[<a href="#ref-Lgm5q6mO" role="doc-biblioref">32</a>]</span>.
We measured corpus similarity by calculating the KL divergence across all three corpora, which focuses on token distribution differences as opposed to token-level differences.</p>
<h3 id="constructing-a-document-representation-for-life-sciences-text">Constructing a Document Representation for Life Sciences Text</h3>
<p>We sought to build a model that would capture the linguistic similarity of articles.
Word2vec is a suite of neural networks designed to model linguistic features of words based on their appearance in text.
These models are trained to either predict a word based on its sentence context as a continuous bag of words (CBOW) or predict the context based on a given word in a skipgram model <span class="citation" data-cites="1GhHIDxuW">[<a href="#ref-1GhHIDxuW" role="doc-biblioref">33</a>]</span>.
Through these prediction tasks the networks learn latent features that can be used for downstream tasks such as identifying similar words.
We used gensim <span class="citation" data-cites="9hB9xc4H">[<a href="#ref-9hB9xc4H" role="doc-biblioref">34</a>]</span> (version 3.8.1) to train a word2vec continuous bag of words (CBOW) <span class="citation" data-cites="1GhHIDxuW">[<a href="#ref-1GhHIDxuW" role="doc-biblioref">33</a>]</span> model over the <em>bioRxiv</em> corpus.
Our neural network architecture had 300 hidden nodes, and we trained this model for 20 epochs.
We set a fixed random seed and used gensim’s default settings for all other hyperparameters.
Following training, we generated a document vector for every article within <em>bioRxiv</em> and PubMed Central.
We calculated the document vector by taking the average of every token present within a given article <span class="citation" data-cites="SdcIDSwR">[<a href="#ref-SdcIDSwR" role="doc-biblioref">35</a>]</span>.
Words absent from the word2vec model were ignored.</p>
<h3 id="visualizing-and-characterizing-preprint-representations">Visualizing and Characterizing Preprint Representations</h3>
<p>We sought to visualize the landscape of preprints and determine the extent to which their representation as document vectors corresponded to author-supplied document labels.
We used principal component analysis (PCA) <span class="citation" data-cites="qkjVrvqt">[<a href="#ref-qkjVrvqt" role="doc-biblioref">36</a>]</span> to project <em>bioRxiv</em> document vectors into a low dimensional space.
We trained this model using the scikit-learn <span class="citation" data-cites="1UyyO5Eg">[<a href="#ref-1UyyO5Eg" role="doc-biblioref">37</a>]</span> implementation of a randomized solver <span class="citation" data-cites="7tMTyalf">[<a href="#ref-7tMTyalf" role="doc-biblioref">38</a>]</span> with a random seed of 100, output of 50 principal components (PCs), and default settings for all other hyperparameters.
After fitting, each preprint has a score for each PC.
To visualize the tokens associated with each PC, we calculated the cosine similarity of each PC to all tokens in our word2vec model’s vocabulary.
We report the top 100 positive and negative scoring tokens in the form of word clouds, where the size of each word corresponds to the magnitude of similarity and color represents positive (orange) or negative (blue) association.</p>
<h3 id="discovering-unannotated-preprint-publication-relationships">Discovering Unannotated Preprint-Publication Relationships</h3>
<p>The <em>bioRxiv</em> maintainers have automated procedures to link preprints to peer reviewed versions and many journals require authors to update preprints with a link to the published version.
However, this automation is largely based on exact matching of certain attributes, and authors can forget to establish a link after publication.
Authors can change the title between a preprint and published version (e.g., <span class="citation" data-cites="1Bpwk7s3x">[<a href="#ref-1Bpwk7s3x" role="doc-biblioref">39</a>]</span> and <span class="citation" data-cites="iBFkqblV">[<a href="#ref-iBFkqblV" role="doc-biblioref">40</a>]</span>), which prevents <em>bioRxiv</em> from automatically establishing a link.
If the authors do not report the publication to <em>bioRxiv</em>, the preprint and the published version are treated as distinct entities despite representing the same underlying research.
We recognized that close proximity in the embedding space could reveal preprint to published version links that were missed by existing automated processes.
First, we used CrossRef <span class="citation" data-cites="XnHb341s">[<a href="#ref-XnHb341s" role="doc-biblioref">41</a>]</span> to identify bioRxiv preprints that were linked to a corresponding published article.
We ignored pairs that contained papers not in the PMCOA corpus.
We calculated the distribution of known preprint to published distances by taking the Euclidean distance between the preprint’s embedding coordinates and the coordinates of its corresponding published version.
We also calculated a background distribution, which consisted of the distance between each preprint with an annotated publication and a randomly selected article from the same journal.
Next, we calculated distances between preprints without a published version link with PubMed Central articles that weren’t matched with a corresponding preprint.
We filtered any potential links with distances that were greater than the minimum value of the background distribution to reduce the curation burden.
Lastly, we binned the remaining pairs based on percentiles from the annotated pairs distribution at the [0,25th percentile), [25th percentile, 50th percentile), [50th percentile, 75th percentile), and [75th percentile, minimum background distance).
We randomly sampled 50 articles from each bin for manual annotation.
We shuffled these four sets to produce a list of 200 potential preprint-published pairs with a randomized order.
We supplied these pairs to two co-authors to manually determine if each link between a preprint and a putative matched version was correct or incorrect.
After the curation process, we encountered eight disagreements between the reviewers.
We supplied the preprint-publication pairs on which reviewers disagreed to a third scientist, who carefully reviewed each case and made a final determination.
We used this curated set to evaluate the extent to which distance in the embedding space revealed true but unannotated links between preprints and their published versions.</p>
<h3 id="measuring-time-duration-for-preprint-publication-process">Measuring Time Duration for Preprint Publication Process</h3>
<p>We measured the time required for preprints to be published in the peer reviewed literature and compared this time within fields and as a function of the extent to which documents changed between the preprint and publication.
We queried bioRxiv’s application programming interface (API) to obtain the date a preprint was posted onto bioRxiv as well as the date a preprint was accepted for publication.
We calculated the difference between the date at which a preprint was first posted and its publication date to provide a publication interval, and we also recorded the number of preprint versions posted onto bioRxiv.
To measure the amount of textual difference, we calculated the Euclidean distance between the document representation of each preprint and the corresponding published version.
We performed linear regression to model the relationship between preprint version count and a preprint’s time to publication as well as the relationship between document representation distances and a preprint’s time to publication.
We visualized results as square bin plots.
We observed a limited number of cases in which authors appeared to post preprints after the date of publication, which results in preprints receiving a negative time difference, as previously reported <span class="citation" data-cites="QrqmHvlC">[<a href="#ref-QrqmHvlC" role="doc-biblioref">42</a>]</span>.
We did not remove preprints that had a negative time publication in our linear regression analysis as it was not strictly necessary, but we removed them in our survival curve analysis where they were incompatible with the analytical approach.
In practice, the number with negative publication times and the short lead time between publication and preprint has a minimal impact on results.</p>
<p>Document distances can be difficult to understand, so we sought to contextualize the meaning of a distance unit.
We selected preprints within the Bioinformatics topic area, which was well-represented on bioRxiv.
For preprints submitted to the Bioinformatics topic area, we sampled a pair of preprints and calculated their differences 1000 times and reported the mean.</p>
<p>In addition to contextualizing the document distance, we also wanted to contextualize differences in the time to publication.
We examined time to publication for each topic area using the Kaplan-Meier estimator <span class="citation" data-cites="PQlBxj25">[<a href="#ref-PQlBxj25" role="doc-biblioref">43</a>]</span> on preprints within bioRxiv, treating preprints not yet published as survival.
We generated these curves using the KaplanMeierFitter function from the lifelines <span class="citation" data-cites="4TsSA7hy">[<a href="#ref-4TsSA7hy" role="doc-biblioref">44</a>]</span> (version 0.25.6) python package.
We reported the half-life of each bioRxiv preprint category.</p>
<h3 id="building-journal-venue-classifiers">Building Journal Venue Classifiers</h3>
<p>We hypothesized that preprints would be more likely to be published in journals that contained similar content to the work in question.
To test this hypothesis, we designed an experiment examining document and journal representations.
First, we removed all journals that had fewer than 100 papers in the PMCOA corpus.
A subset of our PMCOA corpus was directly linked to papers in bioRxiv as they had been published as open access articles.
We held out this subset and treated it as a gold standard test set.
We used the remainder of the PMCOA corpus for training and initial evaluation via cross validation.
We imagined a use case of prioritizing relevant journals for preprint authors, and considered a list of ten journal suggestions to be an appropriate number and we considered a prediction to be a true positive if the correct journal appeared within the ten closest neighbors of the query article.</p>
<p>Certain journals publish articles in a focused topic area, while others publish articles that cover many topics.
Likewise, some journals have a publication rate of at most hundreds of papers per year while others publish at a rate of at least ten-thousand papers per year.
Accounting for these characteristics, we designed two approaches - one centered on manuscripts and another centered on journals.</p>
<p>For the manuscript-centric approach, we identified the ten most similar published manuscripts and evaluated where the documents were published.
We embedded each query article into the space defined by the word2vec model as described for preprints.
We selected the ten manuscripts that were nearest by Euclidean distance in the embedding space and returned the journal in which they were published.
The number of journals returned via this method could be less than ten as multiple papers in close proximity to query article may belong to the same journal.
Because this approach was based on paper proximity, we could return the articles that led to each journal being returned.
However, journals that publish more papers are more frequently recommended in this framing.</p>
<p>For the journal-centric approach, we identified the ten most similar journals by constructing a journal representation in the same embedding space.
We computed journal centroids as the average embedding of all published papers in the journal.
We then project a query article into the same space and return the ten closest journal centroids by Euclidean distance.
This technique guaranteed that at least ten distinct journals were returned and prevented journals that publish many papers from being heavily overrepresented.</p>
<p>In both cases, we set the number of neighbors for each model to be 10 and then evaluated both models via 10-fold cross validation.
We evaluated performance of both classifiers on our gold standard test set of published preprints.</p>
<h3 id="web-application-for-discovering-similar-preprints-and-journals">Web Application for Discovering Similar Preprints and Journals</h3>
<p>We developed a web application that identifies similar papers and journals for any <em>bioRxiv</em> and <em>medRxiv</em> preprint and that places the preprint into the overall document landscape.
Our web application downloads a pdf version of a preprint hosted on the <em>bioRxiv</em> or <em>medRxiv</em> server.
We use pdfminer <span class="citation" data-cites="rF9p7I3q">[<a href="#ref-rF9p7I3q" role="doc-biblioref">45</a>]</span> to extract text from the downloaded pdf.
The extracted text is then fed into our word2vec model to construct a document embedding representation.
We pass this representation onto our journal and manuscript search to identify journals based on the ten closest neighbors of individual papers as well as journal centroids.
We implemented this search using sklearn’s implementation of k-d trees.
To run cost effectively on AWS, we sharded the k-d trees into four trees.</p>
<p>Accompanying these recommendations, we also provide a neural network derived visualization of our training set and the article’s position within it.
We used SAUCIE <span class="citation" data-cites="LxhnWcbN">[<a href="#ref-LxhnWcbN" role="doc-biblioref">46</a>]</span>, an autoencoder designed to cluster single cell RNA-seq data, to build a two-dimensional embedding space that could be applied to newly generated preprints without retraining, a limitation of other approaches that we explored for visualizing entities expected to lie on a nonlinear manifold.
We trained this model on document embeddings of PMCOA articles that did not contain a matching preprint version.
We used the following parameters to train the model: a hidden size of 2, a learning rate of 0.001, lambda_b of 0, lambda_c of 0.001, and lambda_d of 0.001 for 2000 iterations.
When a user requests a new document, we can then project the document on the pre-trained model to generate a visualization in two-dimensional space.
We illustrate our recommendations as a short list and provide access to our network visualization at <a href="https://greenelab.github.io/annorxiver-journal-recommender/">https://greenelab.github.io/annorxiver-journal-recommender/</a>.</p>
<p>We used the fully trained model to project user-requested <em>bioRxiv</em> preprints onto the generated landscape to enable users to see where their preprint falls along the landscape.</p>
<h2 id="results">Results</h2>
<h3 id="comparing-biorxiv-to-other-corpora">Comparing bioRxiv to other corpora</h3>
<h4 id="biorxiv-metadata-statistics">bioRxiv Metadata Statistics</h4>
<p>The preprint landscape is rapidly changing, and the number of bioRxiv preprints in our data download (71,118) was nearly double that of a recent study that reported on a snapshot with 37,648 preprints <span class="citation" data-cites="iPnRRJya">[<a href="#ref-iPnRRJya" role="doc-biblioref">47</a>]</span>.
Because the rate of change is rapid, we first analyzed category data and compared our results with previous findings.
As in previous reports <span class="citation" data-cites="iPnRRJya">[<a href="#ref-iPnRRJya" role="doc-biblioref">47</a>]</span>, neuroscience remains the most common category of preprint followed by bioinformatics (Supplemental Figure <a href="#fig:biorxiv_categories">S1</a>).
Microbiology, which was fifth in the most recent report <span class="citation" data-cites="iPnRRJya">[<a href="#ref-iPnRRJya" role="doc-biblioref">47</a>]</span>, has now surpassed evolutionary biology and genomics to move into third.
When authors upload their preprints, they select from three result category types: new results, confirmatory results or contradictory results.
We found that nearly all preprints (97.5%) were categorized as new results, which is consistent with reports on a smaller set <span class="citation" data-cites="mTE5h91z">[<a href="#ref-mTE5h91z" role="doc-biblioref">48</a>]</span>.
Taken together, the results suggest that while bioRxiv has experienced dramatic growth, the way in which it is being used appears to have remained consistent in recent years.</p>
<h4 id="global-comparison-of-biorxiv-and-pubmed-central">Global Comparison of bioRxiv and PubMed Central</h4>
<div id="tbl:corpora_stats" class="tablenos">
<table>
<caption><span>Table 1:</span> Generated corpora statistics for all corpus used in this project. </caption>
<thead>
<tr class="header">
<th>Metric</th>
<th>bioRxiv</th>
<th>PMC</th>
<th>NYTAC</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>document count</td>
<td>71,118</td>
<td>1,977,647</td>
<td>1,855,658</td>
</tr>
<tr class="even">
<td>sentence count</td>
<td>22,195,739</td>
<td>480,489,811</td>
<td>72,171,037</td>
</tr>
<tr class="odd">
<td>token count</td>
<td>420,969,930</td>
<td>8,597,101,167</td>
<td>1,218,673,384</td>
</tr>
<tr class="even">
<td>stopword count</td>
<td>158,429,441</td>
<td>3,153,077,263</td>
<td>559,391,073</td>
</tr>
<tr class="odd">
<td>avg. document length</td>
<td>312.10</td>
<td>242.96</td>
<td>38.89</td>
</tr>
<tr class="even">
<td>avg. sentence length</td>
<td>22.71</td>
<td>21.46</td>
<td>19.89</td>
</tr>
<tr class="odd">
<td>negatives</td>
<td>1,148,382</td>
<td>24,928,801</td>
<td>7,272,401</td>
</tr>
<tr class="even">
<td>coordinating conjunctions</td>
<td>14,295,736</td>
<td>307,082,313</td>
<td>38,730,053</td>
</tr>
<tr class="odd">
<td>coordinating conjunctions%</td>
<td>3.40%</td>
<td>3.57%</td>
<td>3.18%</td>
</tr>
<tr class="even">
<td>pronouns</td>
<td>4,604,432</td>
<td>74,994,125</td>
<td>46,712,553</td>
</tr>
<tr class="odd">
<td>pronouns%</td>
<td>1.09%</td>
<td>0.87%</td>
<td>3.83%</td>
</tr>
<tr class="even">
<td>passives</td>
<td>15,012,441</td>
<td>342,407,363</td>
<td>19,472,053</td>
</tr>
<tr class="odd">
<td>passive%</td>
<td>3.57%</td>
<td>3.98%</td>
<td>1.60%</td>
</tr>
</tbody>
</table>
</div>
<div id="fig:corpora_comparison_panels," class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/greenelab/annorxiver/472d9b2abd42aecff339fe21c8378f9a556b80d8/figure_generation/output/figure_one_panels.svg" style="width:100.0%" alt="" /><figcaption><span>Figure 1:</span> A. This barplot represents the KL divergence between bioRxiv, Pubmed Central and the reference corpus.
The y-axis is the KL divergence metric where lower values indicates similar distributions and vice versa for higher values.
The x-axis represents the number of highly occuring tokens used to calculate the KL divergence.
B. This is a point range plot of the odds ratio with respect to bioRxiv.
Values greater than one indicate a high association with bioRxiv whereas values less than one indicate high association with PubMed Central.
The dotted line provides a breaking point between both categories.
C. This is a bar chart of token frequency appearing in bioRxiv and PMC respectively.
D. This is a point range plot of the odds ratio with respect to preprints.
Values greater than one indicate a high association with preprints while values less than one indicate a high association with published articles.
The dotted line provides a breaking point between both categories.
E. This is a barchart of token frequency appearing in preprints and published versions of preprints respectively.</figcaption>
</figure>
</div>
<p>The linguistic style of the bioRxiv corpus differs from the PMC corpus.
We compared and contrasted preprints in bioRxiv, published manuscripts in PMC and newspaper articles from the New York Times (NYTAC) against eachother.
We refer to NYTAC as our reference corpus for the following analysis.
We found that bioRxiv is more similar to PMC than to the reference in terms of token frequencies and corpora statistics (Figure ??A and Table <a href="#tbl:corpora_stats">1</a>).
When comparing bioRxiv and PMC to the reference, topic associated and measurement related tokens appear highly enriched (Supplemental Figures <a href="#fig:biorxiv_v_reference">S2</a> and <a href="#fig:pmc_v_reference">S3</a>).
Furthermore, we found that tokens such as “neuron”, “genome”, “RNA” and “network” had a high odds ratio, while tokens such as “patient”, “health”, <span class="math inline">\(\pm\)</span>, and “ml” to have a low odds ratio when comparing bioRxiv to PMC (Figure ??B and ??C).
This separation of tokens suggests that articles focused on clinical trials and patient research are more prevalent in PMC than to bioRxiv.
This separation also suggests that bioRxiv has a predominance of preprints focused on neuroscience and bioinformatic topics.
In regard to writing, citation styles diversify from the familiar “et al.” form as preprints transition through the publication process.
Additionally, published articles have an increase of typesetting (<span class="math inline">\(\pm\)</span>) and measurement symbols (“ml”, “age”) compared to preprints.</p>
<p>A preprint’s linguistic style can change once a preprint has undergone the revision process prior to being published.
We quantified this linguistic difference by calculating the odds ratio of tokens appearing in the union of bioRxiv preprints and their published counterparts within PMC.
Tokens with an odds ratio greater than one are mainly centered on paper/figure references and research specific terms (Figure ??D and ??E).
Tokens with an odds ratio of less than one are focused on data availability, and research measurements such as number of cases and controls or significance testing (Figure ??D and ??E).
This enrichment suggests that a key piece in the publication process is verifying that essential parts of research (e.g. data availability, specific measurements) are obvious to future readers within the scientific community.</p>
<h3 id="topic-analysis-of-biorxivs-principal-components">Topic Analysis of bioRxiv’s Principal Components</h3>
<div id="fig:topic_analysis_panels" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/greenelab/annorxiver/472d9b2abd42aecff339fe21c8378f9a556b80d8/figure_generation/output/figure_two_panels.svg" style="width:100.0%" alt="" /><figcaption><span>Figure 2:</span> A. The scatter plot at the top right (A) is a visualization of documents being plotted along the PC directions.
Article categories were hand-picked based on the concepts captured by each PC.
B. The word cloud depicts the cosine similarity score between tokens and the second PC.
Tokens in orange are most similar to a PC’s positive direction while tokens in blue are most similar to a PC’s negative direction.
The size of each token indicates the magnitude of the similarity score.
C. The word cloud depicts the cosine similarity score between tokens and the first PC.
Tokens in orange are most similar to a PC’s positive direction while tokens in blue are most similar to a PC’s negative direction.
The size of each token indicates the magnitude of the similarity score.
D. This is box plot shows preprints in each article category projected along the PC1 direction.
Negative values indicate molecular biology concepts, while positive values indicate quantitative biology concepts.
E. This is box plot shows preprints in each article category projected along the PC2 direction.
Negative values indicate neuroscience concepts, while positive values indicate bioinformatic concepts.</figcaption>
</figure>
</div>
<p>We explored the primary differences between the full text of bioRxiv preprints by performing principal components analysis on generated document embeddings.
We visualized the correspondence between tokens and the loadings for each principal component (Figure <a href="#fig:topic_analysis_panels">2</a>).
We also visualized documents projected on selected principal components (Figure <a href="#fig:topic_analysis_panels">2</a>A).
The first principal component separates bioRxiv preprints that encompass molecular biology results with preprints that contain quantitative biology results (Figure <a href="#fig:topic_analysis_panels">2</a>B).
This highlights the bisection of biomedical research where majority of results can be categorized under the molecular biology category or the quantitative biology category.
Furthermore, this bisecting trend is evident across individual preprint categories as most categories lie on either side of the first principal component (Figure <a href="#fig:topic_analysis_panels">2</a>D).
As with the first principal component we also provide example preprints from the systems biology category to reinforce this concept (Supplemental Table <a href="#tbl:five_pc1_table">2</a>).</p>
<p>The second principal component represents the concept of neuroscience vs bioinformatics (Figure <a href="#fig:topic_analysis_panels">2</a>C).
This principal component suggests that the bulk of preprints within bioRxiv are largely focused around neuroscience and bioinformatic concepts.
This split is evident in Figure <a href="#fig:topic_analysis_panels">2</a>E as enriched categories along this principal component are quite related to neuroscience (negative end) or bioinformatics (positive end).
We provide example preprints from the systems biology category to reinforce this concept (Supplemental Table <a href="#tbl:five_pc2_table">3</a>).
More principal component word clouds can be found on our journal recommender website and within our online repository (see Software and Data Availability).</p>
<h3 id="identifying-preprints-that-were-not-linked-with-their-corresponding-publications">Identifying preprints that were not linked with their corresponding publications</h3>
<div id="fig:preprint_links_panels" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/f9d8861e3d60afa878c5d0088b7502adae15ebe3/figure_generation/output/figure_three_panels.svg" style="width:100.0%" alt="" /><figcaption><span>Figure 3:</span> A. This violin plot shows the distribution of distances between the preprint-corresponding published version category and the preprint-randomly sampled articles category.
B. This bar chart depicts the fraction of true positives over the total number of pairs in each bin.
Each bin contains a total of 200 annotated pairs and is based on the percentiles of the preprint-published distribution.
C. This line plot shows the publication rate of preprints since bioRxiv first started.
The x-axis represents months since bioRxiv started and the y-axis represents the proportion of preprints published.
The light blue line represents the publication rate estimated by Abdill et al. <span class="citation" data-cites="iPnRRJya">[<a href="#ref-iPnRRJya" role="doc-biblioref">47</a>]</span>.
The dark blue line represents the updated publication rate without missing links added while the dark green line is the updated publication rate with missing links added.
The horizontal lines represent the overall proportion of preprints that are published.</figcaption>
</figure>
</div>
<p>Many journals require that authors update preprints with links to the published version of their article.
This is accomplished in two ways: <em>bioRxiv</em> may detect the link and automatically add it or authors may notify <em>bioRxiv</em> that their preprint was published.
Sproadically, there are cases where <em>bioRxiv</em> may miss detecting a link or authors may forget to notiy <em>bioRxiv</em> of their recent publcation.
These missing links can make it more difficult to identify the latest version of scientific manuscripts and estimate the fraction of articles that are eventually published <span class="citation" data-cites="iPnRRJya">[<a href="#ref-iPnRRJya" role="doc-biblioref">47</a>]</span>.
We used distance in the document space to identify preprints without an annotated publication but contained very similar content to published articles.
We found that distances between preprints and their corresponding published versions were lower than preprints paired with a random article published in the same journal (Figure <a href="#fig:preprint_links_panels">3</a>A).
This observation suggests that pairs with low embedding distances could be considered a true match, so we separated articles into quantiles based on the distribution of distances between true preprint-publication pairs.
We curated 50 potential preprint-publication pairs from each of four quantiles and achieved a high inter-rater reliability of 91.7% (Cohen’s Kappa <span class="citation" data-cites="oeGSw37I">[<a href="#ref-oeGSw37I" role="doc-biblioref">49</a>]</span>) for this task.
Out of these two hundred pairs we found that approximately 98% of pairs with an embedding distance in the 0-25th and 25th-50th percentile bins were true matches (Figure <a href="#fig:preprint_links_panels">3</a>B).
These two bins contained 1,720 preprint-article pairs, suggesting that many preprints have been published but not previously connected with their published versions.</p>
<p>We overlaid these new annotations onto existing annotations to reassess the overall preprint publication rate reported by Abdill et al. <span class="citation" data-cites="iPnRRJya">[<a href="#ref-iPnRRJya" role="doc-biblioref">47</a>]</span>.
Our filtering criteria were intentionally stringent, so the increased estimate of publication rate amounts to a few percent (Figure <a href="#fig:preprint_links_panels">3</a>C).
Many of these missed annotations were for preprints posted in the 2017-2018 interval.
Compared to preprints published in 2019 and later, the preprints posted in 2017-2018 are old enough to have a high chance of being published; however, it is interesting that the rate for older preprints was not observed to be higher.</p>
<h3 id="factors-that-affect-the-time-between-preprinting-and-publication">Factors that affect the time between preprinting and publication</h3>
<div id="fig:publication_delay_panels" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/232ab6f9f29f283070ed08b88a421b3bedaf5cbd/figure_generation/output/figure_four_panels.svg" style="width:100.0%" alt="" /><figcaption><span>Figure 4:</span> A.This hexbin plot depicts the amount of time it takes a preprint to be published against the distances of a preprint’s first version and its corresponding published version.
The x-axis represents the Euclidean distance between document representations, while the y-axis represents the number of days elapsed between a preprint posted on bioRxiv and the time a preprint is published.
The color bar on the right represents the density of each hexbin in this plot where more dense regions have a brighter color compared to their counterparts.
B.This violin plot depicts the amount of time it takes a preprint to be published against the number of versions posted for a specific preprint.
The x-axis represents the number of different versions a preprint has on bioRxiv, while the y-axis represents the number of days elapsed between a preprint posted on bioRxiv and the time a preprint is published.
Inside each violin is a boxplot for the respective preprint version distribution.
C.This bargraph depicts the amount of time it takes to get half of the total number of preprints published.
The x-axis reprints days until 50% of preprints are published and the y-axis reprints the different preprint categories.
The error bars represent 95% confidence intervals for each preprint’s category half-life.</figcaption>
</figure>
</div>
<p>Preprints undergo multiple review checkpoints before they are published within a journal <span class="citation" data-cites="TMxQYqVE">[<a href="#ref-TMxQYqVE" role="doc-biblioref">50</a>]</span>.
Oftentimes these checkpoints may result in rejection or revisions requested by a reviewer <span class="citation" data-cites="TMxQYqVE">[<a href="#ref-TMxQYqVE" role="doc-biblioref">50</a>]</span>.
These negative outcomes result in authors may having to drastically edit their preprint, which greatly impedes a preprint reaching a published endpoint.
We sought to quantify the extent to which preprints are stalled when faced with a setback from the peer-review process.
On average preprints are delayed approximately 16 days for every distance unit change (Figure <a href="#fig:publication_delay_panels">4</a>A).
We found that the average distance between two preprints’ in the bioinformatics category was 5.068, which suggests that a single distance unit represents a fifth of a preprint’s total text being changed.
Sometimes preprints have to undergo drastic revisions that result in a new version being created.
We found that on average it takes 51 days for authors to construct a new version of a preprint (Figure <a href="#fig:publication_delay_panels">4</a>B).
Both the document distance trend and the version number trend confirm that the larger the revision the longer it takes for a preprint to be published.</p>
<p>Preprints in certain categories take less time to publish than others.
we sought to quantify the time each category takes to publish half their total number of preprints.
Every preprint category takes at least 124 days to publish half of their respective preprints (Figure <a href="#fig:publication_delay_panels">4</a>C).
Categories that took the least amount of time were microbiology and zoology, while scientific communication and education took the most time (Figure <a href="#fig:publication_delay_panels">4</a>C).
Overall, this suggests that preprints in the microbiology and zoology categories may face less peer-review setbacks compared to other categories.</p>
<h3 id="recommending-journals-based-on-preprint-representation">Recommending Journals Based on Preprint Representation</h3>
<p>We sought to identify journals that might publish a preprint based on the text of a paper.
We trained two different classifiers to predict the journal endpoints for already published papers.
One classifier uses the nearest journal centroids, which attempts to capture the topic area of a journal.
The other classifier aims to be more granular and recommends journals based on close proximity of individual papers.
Both classifiers achieved a substantial increase over the random baseline; however, our predictors are not perfect (Supplemental Figure <a href="#fig:knn_auc">S4</a>).
This is expected as our dataset contains 2516 different journals where some journals publish papers that cover very specific topic while others publish papers that have a broad set of covered topics.
Our journal centroid classifier performed better than the nearest paper classifier on the held out test set (Supplemental Figure <a href="#fig:knn_auc">S4</a>).
Overall, our software provides a starting point for authors to use the text of their preprints to identify potentially suitable publication venues.</p>
<div id="fig:journal_rec_workflow" class="fignos">
<figure>
<img src="images/journal_recommender_workflow.png" style="width:100.0%" alt="" /><figcaption><span>Figure 5:</span> Here is the workflow of the journal recommender web-app.
Starting with the homescreen users can paste in a <em>bioRxiv</em> or <em>medRxiv</em> DOI, which sends a request to biorxiv or medrxiv (A).
Next our app preprocesses the preprint and returns a listing of the top ten most similar papers (B) and the top ten closest journals to the query (C).
Following the listing, our app manually plots the preprint query onto the Pubmed Central Landscape (D).
Lastly, users can click on a square within the landscape, which will show bin statistics as well as associated word-odd ratios (E).</figcaption>
</figure>
</div>
<p>We constructed an online app that provides users with journal suggestions based on their preprint content.
Users supply DOIs from <em>bioRxiv</em> or <em>medRxiv</em>.
The application then downloads the article, converts the PDF to text, calculates a document embedding score, and returns the ten papers and journals with the most similar representations in the embedding space.
It also embeds the document into the overall PMC landscape for visualization and allows the user to examine principal components and term enrichment for each bin within the landscape (Figure <a href="#fig:journal_rec_workflow">5</a>).</p>
<h2 id="discussion">Discussion</h2>
<p>We analyzed the language contained used in preprints and examined how it changes through the publication process.
We found that bioRxiv and PubMed Central (PMC) have similar word frequency distributions, which suggests that the overall manner of writing is consistent with the biomedical literature.
At the token level, those most strongly associated with bioRxiv are related to neuroscience and bioinformatics, which are also fields that have seen high uptake of preprinting <span class="citation" data-cites="iPnRRJya">[<a href="#ref-iPnRRJya" role="doc-biblioref">47</a>]</span>.
We noticed that a multitude of preprints highly associated with the first principal component have restrictive or no copyright license (Supplemental Table <a href="#tbl:five_pc1_table">2</a>).
This finding highlights the ongoing problem of restricted access within the scientific community <span class="citation" data-cites="1FHYUCO1v 11GpiE0CN">[<a href="#ref-1FHYUCO1v" role="doc-biblioref">51</a>,<a href="#ref-11GpiE0CN" role="doc-biblioref">52</a>]</span>.
We also found that the second principal component for our language embedding differentiated neuroscience and bioinformatics papers.</p>
<p>We examined preprints that were textually similar to published articles and found numerous missing links between preprints and their publsihed counterparts.
This observation led us to find that the life sciences preprint publication rate is higher than previously estimated (Figure ??).
Preprint-publication similarity also predicts journal endpoints with modest performance for already published articles.
This observation enabled us to provide a web application that allows users to identify the papers and journals that are most similar to a <em>bioRxiv</em> or <em>medRxiv</em> preprint.</p>
<h2 id="conclusion-and-future-directions">Conclusion and Future Directions</h2>
<p>Our linguistic analysis did not reveal substantial changes in the language during the peer-reviewed publishing process.
The tokens most strongly associated with the peer reviewed form, as opposed to the preprint form, were associated with data availability and statistical reporting.
We found that embeddings of preprints and publications could be compared and that distance in this space was meaningful in terms of topic area and the journal of eventual publication.
Being able to identify similar preprints and publications using text content makes it feasible to begin tackling more detailed questions, and our analytical software is all open source to enable others to build upon them.
The analysis of preprints’ full text can support new tools that accelerate publishing, integrity checks, and other critically important contributions.</p>
<h2 id="software-and-data-availability">Software and Data Availability</h2>
<p>An online version of this manuscript is available under a Creative Commons Attribution License at <a href="https://greenelab.github.io/annorxiver_manuscript/">https://greenelab.github.io/annorxiver_manuscript/</a>.
Source for the research portions of this project is dual licensed under the BSD 3-Clause and Creative Commons Public Domain Dedication Licenses at <a href="https://github.com/greenelab/annorxiver">https://github.com/greenelab/annorxiver</a>.
The journal recommendation website can be found at <a href="https://greenelab.github.io/annorxiver-journal-recommender/">https://greenelab.github.io/annorxiver-journal-recommender/</a> and code for the website is available under a BSD-2-Clause Plus Patent License at <a href="https://github.com/greenelab/annorxiver-journal-recommender">https://github.com/greenelab/annorxiver-journal-recommender</a>.
Full text access for the bioRxiv repository is available at <a href="https://www.biorxiv.org/tdm">https://www.biorxiv.org/tdm</a>.
Access to PubMed Central’s Open Access subset is available on NCBI’s FTP server at <a href="https://www.ncbi.nlm.nih.gov/pmc/tools/ftp/">https://www.ncbi.nlm.nih.gov/pmc/tools/ftp/</a>.
Access to the New York Times Annotated Corpus (NYTAC) is available upon request with the Linguistic Data Consortium at <a href="https://catalog.ldc.upenn.edu/LDC2008T19">https://catalog.ldc.upenn.edu/LDC2008T19</a>.</p>
<h2 id="acknowledgements">Acknowledgements</h2>
<p>The authors would like to thank Ariel Hippen Anderson for evaluating potential missing preprint to published version links.
We also would like to thank Richard Sever and the <em>bioRxiv</em> team for their assistance with access to and support with questions about preprint full text downloaded from <em>bioRxiv</em>.
This work was supported by <a href="https://www.moore.org/grant-detail?grantId=GBMF4552">Grant GBMF4552</a> from the Gordon Betty Moore Foundation and by NIH T32HG00046, Computational Genomics training grant, from the National Human Genome Reserach Institute (NHGRI).</p>
<h2 id="competing-interest">Competing Interest</h2>
<p>Need to note here.</p>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-ePmdRbmq">
<p>1. <strong>The prehistory of biology preprints: A forgotten experiment from the 1960s</strong> <br />
Matthew Cobb<br />
<em>PLOS Biology</em> (2017-11-16) <a href="https://doi.org/c6wv">https://doi.org/c6wv</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pbio.2003995">10.1371/journal.pbio.2003995</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29145518">29145518</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5690419">PMC5690419</a></p>
</div>
<div id="ref-lN9ljhid">
<p>2. <strong>Preprint Déjà Vu</strong> <br />
Paul Ginsparg<br />
<em>The EMBO Journal</em> (2016-10-19) <a href="https://doi.org/f3r9vf">https://doi.org/f3r9vf</a> <br />
DOI: <a href="https://doi.org/10.15252/embj.201695531">10.15252/embj.201695531</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27760783">27760783</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5167339">PMC5167339</a></p>
</div>
<div id="ref-i9BKPuqb">
<p>3. <strong>bioRxiv: the preprint server for biology</strong> <br />
Richard Sever, Ted Roeder, Samantha Hindle, Linda Sussman, Kevin-John Black, Janet Argentine, Wayne Manos, John R. Inglis<br />
<em>bioRxiv</em> (2019-11-06) <a href="https://doi.org/ggc46z">https://doi.org/ggc46z</a> <br />
DOI: <a href="https://doi.org/10.1101/833400">10.1101/833400</a></p>
</div>
<div id="ref-QHX7Bezs">
<p>4. <strong>Abstract</strong> <br />
eLife Sciences Publications, Ltd<br />
(2019-05-09) <a href="https://doi.org/gf5cqt">https://doi.org/gf5cqt</a> <br />
DOI: <a href="https://doi.org/10.7554/elife.45133.001">10.7554/elife.45133.001</a></p>
</div>
<div id="ref-PmF48i62">
<p>5. <strong>Biologists urged to hug a preprint</strong> <br />
Ewen Callaway, Kendall Powell<br />
<em>Nature</em> (2016-02-16) <a href="https://doi.org/ghdd62">https://doi.org/ghdd62</a> <br />
DOI: <a href="https://doi.org/10.1038/530265a">10.1038/530265a</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26887471">26887471</a></p>
</div>
<div id="ref-1EwmaUwf9">
<p>6. <strong>Peer Review and bioRxiv</strong> <br />
Leslie M. Loew<br />
<em>Biophysical Journal</em> (2016-08) <a href="https://doi.org/ghdd6x">https://doi.org/ghdd6x</a> <br />
DOI: <a href="https://doi.org/10.1016/j.bpj.2016.06.035">10.1016/j.bpj.2016.06.035</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27508451">27508451</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4982934">PMC4982934</a></p>
</div>
<div id="ref-XqZPlHz7">
<p>7. <strong>Preprints for the life sciences</strong> <br />
J. M. Berg, N. Bhalla, P. E. Bourne, M. Chalfie, D. G. Drubin, J. S. Fraser, C. W. Greider, M. Hendricks, C. Jones, R. Kiley, … C. Wolberger<br />
<em>Science</em> (2016-05-19) <a href="https://doi.org/bmp7">https://doi.org/bmp7</a> <br />
DOI: <a href="https://doi.org/10.1126/science.aaf9133">10.1126/science.aaf9133</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27199406">27199406</a></p>
</div>
<div id="ref-IyRO6uC">
<p>8. <strong>The rise of preprints in chemistry</strong> <br />
François-Xavier Coudert<br />
<em>Nature Chemistry</em> (2020-05-18) <a href="https://doi.org/ghdd64">https://doi.org/ghdd64</a> <br />
DOI: <a href="https://doi.org/10.1038/s41557-020-0477-5">10.1038/s41557-020-0477-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32424256">32424256</a></p>
</div>
<div id="ref-Cy352E90">
<p>9. <strong>Preprints: An underutilized mechanism to accelerate outbreak science</strong> <br />
Michael A. Johansson, Nicholas G. Reich, Lauren Ancel Meyers, Marc Lipsitch<br />
<em>PLOS Medicine</em> (2018-04-03) <a href="https://doi.org/gg922h">https://doi.org/gg922h</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pmed.1002549">10.1371/journal.pmed.1002549</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29614073">29614073</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5882117">PMC5882117</a></p>
</div>
<div id="ref-jXP4z24U">
<p>10. <strong>On the value of preprints: An early career researcher perspective</strong> <br />
Sarvenaz Sarabipour, Humberto J. Debat, Edward Emmott, Steven J. Burgess, Benjamin Schwessinger, Zach Hensel<br />
<em>PLOS Biology</em> (2019-02-21) <a href="https://doi.org/gfw8hd">https://doi.org/gfw8hd</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pbio.3000151">10.1371/journal.pbio.3000151</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30789895">30789895</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6400415">PMC6400415</a></p>
</div>
<div id="ref-YmvKU4HJ">
<p>11. <strong>In praise of preprints</strong> <br />
Norman K. Fry, Helina Marshall, Tasha Mellins-Cohen<br />
<em>Microbial Genomics</em> (2019-04-01) <a href="https://doi.org/gg3bxc">https://doi.org/gg3bxc</a> <br />
DOI: <a href="https://doi.org/10.1099/mgen.0.000259">10.1099/mgen.0.000259</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30938670">30938670</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6521583">PMC6521583</a></p>
</div>
<div id="ref-ZTyQYlDz">
<p>12. <strong>arXiv.org: the Los Alamos National Laboratory e‐print server</strong> <br />
Gerry McKiernan<br />
<em>International Journal on Grey Literature</em> (2000-09) <a href="https://doi.org/fg8pw7">https://doi.org/fg8pw7</a> <br />
DOI: <a href="https://doi.org/10.1108/14666180010345564">10.1108/14666180010345564</a></p>
</div>
<div id="ref-NKHNs9j3">
<p>13. <strong>medRxiv.org - the preprint server for Health Sciences</strong> <a href="https://www.medrxiv.org/">https://www.medrxiv.org/</a></p>
</div>
<div id="ref-lMj4YzIP">
<p>14. <strong>The Second Wave of Preprint Servers: How Can Publishers Keep Afloat?</strong> <br />
By<br />
<em>The Scholarly Kitchen</em> (2019-10-16) <a href="https://scholarlykitchen.sspnet.org/2019/10/16/the-second-wave-of-preprint-servers-how-can-publishers-keep-afloat/">https://scholarlykitchen.sspnet.org/2019/10/16/the-second-wave-of-preprint-servers-how-can-publishers-keep-afloat/</a></p>
</div>
<div id="ref-1DCEzEMkZ">
<p>15. <strong>Rxivist.org: Sorting biology preprints using social media and readership metrics</strong> <br />
Richard J. Abdill, Ran Blekhman<br />
<em>PLOS Biology</em> (2019-05-21) <a href="https://doi.org/dm27">https://doi.org/dm27</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pbio.3000269">10.1371/journal.pbio.3000269</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31112533">31112533</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6546241">PMC6546241</a></p>
</div>
<div id="ref-PJwR7lcq">
<p>16. <strong>How the Scientific Community Reacts to Newly Submitted Preprints: Article Downloads, Twitter Mentions, and Citations</strong> <br />
Xin Shuai, Alberto Pepe, Johan Bollen<br />
<em>PLoS ONE</em> (2012-11-01) <a href="https://doi.org/f4cw6r">https://doi.org/f4cw6r</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pone.0047523">10.1371/journal.pone.0047523</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23133597">23133597</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3486871">PMC3486871</a></p>
</div>
<div id="ref-13dwNbYs7">
<p>17. <strong>The relationship between bioRxiv preprints, citations and altmetrics</strong> <br />
Nicholas Fraser, Fakhri Momeni, Philipp Mayr, Isabella Peters<br />
<em>Quantitative Science Studies</em> (2020-04-01) <a href="https://doi.org/gg2cz3">https://doi.org/gg2cz3</a> <br />
DOI: <a href="https://doi.org/10.1162/qss_a_00043">10.1162/qss_a_00043</a></p>
</div>
<div id="ref-49GeQtua">
<p>18. <strong>Releasing a preprint is associated with more attention and citations for the peer-reviewed article</strong> <br />
Darwin Y Fu, Jacob J Hughey<br />
<em>eLife</em> (2019-12-06) <a href="https://doi.org/ghd3mv">https://doi.org/ghd3mv</a> <br />
DOI: <a href="https://doi.org/10.7554/elife.52646">10.7554/elife.52646</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31808742">31808742</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6914335">PMC6914335</a></p>
</div>
<div id="ref-tbWPLbOF">
<p>19. <strong>Quantifying and contextualizing the impact of bioRxiv preprints through automated social media audience segmentation</strong> <br />
Jedidiah Carlson, Kelley Harris<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-10) <a href="https://doi.org/ghdd66">https://doi.org/ghdd66</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.06.981589">10.1101/2020.03.06.981589</a></p>
</div>
<div id="ref-LcHjf487">
<p>20. <strong>An analysis of published journals for papers posted on bioR <sub>X</sub> iv</strong> <br />
Hiroyuki Tsunoda, Yuan Sun, Masaki Nishizawa, Xiaomin Liu, Kou Amano<br />
<em>Proceedings of the Association for Information Science and Technology</em> (2019-10-18) <a href="https://doi.org/ggz7f9">https://doi.org/ggz7f9</a> <br />
DOI: <a href="https://doi.org/10.1002/pra2.175">10.1002/pra2.175</a></p>
</div>
<div id="ref-tTgEr7g1">
<p>21. <strong>The Need for Speed: How Quickly Do Preprints Become Published Articles?</strong> <br />
Rachel Herbert, Kate Gasson, Alex Ponsford<br />
<em>SSRN Electronic Journal</em> (2019) <a href="https://doi.org/ghd3mt">https://doi.org/ghd3mt</a> <br />
DOI: <a href="https://doi.org/10.2139/ssrn.3455146">10.2139/ssrn.3455146</a></p>
</div>
<div id="ref-9gVmEi52">
<p>22. <strong>Machine access and text/data mining resources | bioRxiv</strong> <a href="https://www.biorxiv.org/tdm">https://www.biorxiv.org/tdm</a></p>
</div>
<div id="ref-u294RvPz">
<p>23. <strong>PubMed Central: The GenBank of the published literature</strong> <br />
R. J. Roberts<br />
<em>Proceedings of the National Academy of Sciences</em> (2001-01-16) <a href="https://doi.org/bbn9k8">https://doi.org/bbn9k8</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.98.2.381">10.1073/pnas.98.2.381</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11209037">11209037</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33354">PMC33354</a></p>
</div>
<div id="ref-iyblQx2M">
<p>24. <strong>Gold open access: the best of both worlds</strong> <br />
M. A. G. van der Heyden, T. A. B. van Veen<br />
<em>Netherlands Heart Journal</em> (2017-12-01) <a href="https://doi.org/ggzfr9">https://doi.org/ggzfr9</a> <br />
DOI: <a href="https://doi.org/10.1007/s12471-017-1064-2">10.1007/s12471-017-1064-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29196877">29196877</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5758455">PMC5758455</a></p>
</div>
<div id="ref-lBh7pyRc">
<p>25. <strong>How Papers Get Into PMC</strong> <a href="https://www.ncbi.nlm.nih.gov/pmc/about/submission-methods/">https://www.ncbi.nlm.nih.gov/pmc/about/submission-methods/</a></p>
</div>
<div id="ref-vGaAJBqg">
<p>26. <strong>8.2.2 NIH Public Access Policy</strong> <a href="https://grants.nih.gov/grants/policy/nihgps/html5/section_8/8.2.2_nih_public_access_policy.htm">https://grants.nih.gov/grants/policy/nihgps/html5/section_8/8.2.2_nih_public_access_policy.htm</a></p>
</div>
<div id="ref-rzmpIYmn">
<p>27. <strong>PMC Overview</strong> <a href="https://www.ncbi.nlm.nih.gov/pmc/about/intro/">https://www.ncbi.nlm.nih.gov/pmc/about/intro/</a></p>
</div>
<div id="ref-rURwqquJ">
<p>28. <strong>PMC text mining subset in BioC: about three million full-text articles and growing</strong> <br />
Donald C Comeau, Chih-Hsuan Wei, Rezarta Islamaj Doğan, Zhiyong Lu<br />
<em>Bioinformatics</em> (2019-09-15) <a href="https://doi.org/ggzfsb">https://doi.org/ggzfsb</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btz070">10.1093/bioinformatics/btz070</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30715220">30715220</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6748740">PMC6748740</a></p>
</div>
<div id="ref-18ZyyTcTe">
<p>29. <strong>PubTator central: automated concept annotation for biomedical full text articles</strong> <br />
Chih-Hsuan Wei, Alexis Allot, Robert Leaman, Zhiyong Lu<br />
<em>Nucleic Acids Research</em> (2019-07-02) <a href="https://doi.org/ggzfsc">https://doi.org/ggzfsc</a> <br />
DOI: <a href="https://doi.org/10.1093/nar/gkz389">10.1093/nar/gkz389</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31114887">31114887</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6602571">PMC6602571</a></p>
</div>
<div id="ref-18JcY2okY">
<p>30. <strong>The new york times annotated corpus</strong> <br />
Evan Sandhaus<br />
<em>Linguistic Data Consortium, Philadelphia</em> (2008)</p>
</div>
<div id="ref-q2fFAZTG">
<p>31. <strong>spaCy 2: Natural language understanding with Bloom embeddings, convolutional neural networks and incremental parsing</strong> <br />
Matthew Honnibal, Ines Montani<br />
(2017)</p>
</div>
<div id="ref-Lgm5q6mO">
<p>32. <strong>Odds Ratio</strong> <br />
Steven Tenny, Mary R. Hoffman<br />
<em>StatPearls</em> (2020) <a href="http://www.ncbi.nlm.nih.gov/books/NBK431098/">http://www.ncbi.nlm.nih.gov/books/NBK431098/</a></p>
</div>
<div id="ref-1GhHIDxuW">
<p>33. <strong>Efficient Estimation of Word Representations in Vector Space</strong> <br />
Tomas Mikolov, Kai Chen, Greg Corrado, Jeffrey Dean<br />
<em>arXiv</em> (2013-09-10) <a href="https://arxiv.org/abs/1301.3781">https://arxiv.org/abs/1301.3781</a></p>
</div>
<div id="ref-9hB9xc4H">
<p>34. <strong>Software Framework for Topic Modelling with Large Corpora</strong> <br />
Radim Řehůřek, Petr Sojka<br />
<em>Proceedings of the LREC 2010 Workshop on New Challenges for NLP Frameworks</em> (2010-05-22)</p>
</div>
<div id="ref-SdcIDSwR">
<p>35. <strong>Distributed Representations of Sentences and Documents</strong> <br />
Quoc V. Le, Tomas Mikolov<br />
<em>arXiv</em> (2014-05-26) <a href="https://arxiv.org/abs/1405.4053">https://arxiv.org/abs/1405.4053</a></p>
</div>
<div id="ref-qkjVrvqt">
<p>36. <strong>Probabilistic Principal Component Analysis</strong> <br />
Michael E. Tipping, Christopher M. Bishop<br />
<em>Journal of the Royal Statistical Society: Series B (Statistical Methodology)</em> (1999-08) <a href="https://doi.org/b3hjwt">https://doi.org/b3hjwt</a> <br />
DOI: <a href="https://doi.org/10.1111/1467-9868.00196">10.1111/1467-9868.00196</a></p>
</div>
<div id="ref-1UyyO5Eg">
<p>37. <strong>Scikit-learn: Machine learning in Python</strong> <br />
F. Pedregosa, G. Varoquaux, A. Gramfort, V. Michel, B. Thirion, O. Grisel, M. Blondel, P. Prettenhofer, R. Weiss, V. Dubourg, … E. Duchesnay<br />
<em>Journal of Machine Learning Research</em> (2011)</p>
</div>
<div id="ref-7tMTyalf">
<p>38. <strong>Finding structure with randomness: Probabilistic algorithms for constructing approximate matrix decompositions</strong> <br />
Nathan Halko, Per-Gunnar Martinsson, Joel A. Tropp<br />
<em>arXiv</em> (2014-04-29) <a href="https://arxiv.org/abs/0909.4061">https://arxiv.org/abs/0909.4061</a></p>
</div>
<div id="ref-1Bpwk7s3x">
<p>39. <strong>The <em>Drosophila</em> Cortactin Binding Protein 2 homolog, Nausicaa, regulates lamellipodial actin dynamics in a Cortactin-dependent manner</strong> <br />
Meghan E. O’Connell, Divya Sridharan, Tristan Driscoll, Ipsita Krishnamurthy, Wick G. Perry, Derek A. Applewhite<br />
<em>bioRxiv</em> (2018-07-24) <a href="https://doi.org/gg4hp7">https://doi.org/gg4hp7</a> <br />
DOI: <a href="https://doi.org/10.1101/376665">10.1101/376665</a></p>
</div>
<div id="ref-iBFkqblV">
<p>40. <strong>The <em>Drosophila</em> protein, Nausicaa, regulates lamellipodial actin dynamics in a Cortactin-dependent manner</strong> <br />
Meghan E. O’Connell, Divya Sridharan, Tristan Driscoll, Ipsita Krishnamurthy, Wick G. Perry, Derek A. Applewhite<br />
<em>Biology Open</em> (2019-06-15) <a href="https://doi.org/gg4hp8">https://doi.org/gg4hp8</a> <br />
DOI: <a href="https://doi.org/10.1242/bio.038232">10.1242/bio.038232</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31164339">31164339</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6602326">PMC6602326</a></p>
</div>
<div id="ref-XnHb341s">
<p>41. <strong>CrossRef Text and Data Mining Services</strong> <br />
Rachael Lammey<br />
<em>Insights the UKSG journal</em> (2015-07-07) <a href="https://doi.org/gg4hp9">https://doi.org/gg4hp9</a> <br />
DOI: <a href="https://doi.org/10.1629/uksg.233">10.1629/uksg.233</a></p>
</div>
<div id="ref-QrqmHvlC">
<p>42. <strong>Medium – Where good ideas find you.</strong> <br />
Medium<br />
<a href="https://medium.com">https://medium.com</a></p>
</div>
<div id="ref-PQlBxj25">
<p>43. <strong>Understanding survival analysis: Kaplan-Meier estimate</strong> <br />
Jugal Kishore, ManishKumar Goel, Pardeep Khanna<br />
<em>International Journal of Ayurveda Research</em> (2010) <a href="https://doi.org/fdft75">https://doi.org/fdft75</a> <br />
DOI: <a href="https://doi.org/10.4103/0974-7788.76794">10.4103/0974-7788.76794</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21455458">21455458</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3059453">PMC3059453</a></p>
</div>
<div id="ref-4TsSA7hy">
<p>44. <strong>CamDavidsonPilon/lifelines: v0.25.6</strong> <br />
Cameron Davidson-Pilon, Jonas Kalderstam, Noah Jacobson, Sean-Reed, Ben Kuhn, Paul Zivich, Mike Williamson, AbdealiJK, Deepyaman Datta, Andrew Fiore-Gartland, … Jlim13<br />
<em>Zenodo</em> (2020-10-26) <a href="https://doi.org/ghh2d3">https://doi.org/ghh2d3</a> <br />
DOI: <a href="https://doi.org/10.5281/zenodo.4136578">10.5281/zenodo.4136578</a></p>
</div>
<div id="ref-rF9p7I3q">
<p>45. <strong>Welcome to pdfminer.six’s documentation! — pdfminer.six 20201018 documentation</strong> <a href="https://pdfminersix.readthedocs.io/en/latest/index.html">https://pdfminersix.readthedocs.io/en/latest/index.html</a></p>
</div>
<div id="ref-LxhnWcbN">
<p>46. <strong>Assessing the Heterogeneity of Cardiac Non-myocytes and the Effect of Cell Culture with Integrative Single Cell Analysis</strong> <br />
Brian S. Iskra, Logan Davis, Henry E. Miller, Yu-Chiao Chiu, Alexander R. Bishop, Yidong Chen, Gregory J. Aune<br />
<em>Cold Spring Harbor Laboratory</em> (2020-03-05) <a href="https://doi.org/gg9353">https://doi.org/gg9353</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.03.04.975177">10.1101/2020.03.04.975177</a></p>
</div>
<div id="ref-iPnRRJya">
<p>47. <strong>Tracking the popularity and outcomes of all bioRxiv preprints</strong> <br />
Richard J Abdill, Ran Blekhman<br />
<em>eLife</em> (2019-04-24) <a href="https://doi.org/gf2str">https://doi.org/gf2str</a> <br />
DOI: <a href="https://doi.org/10.7554/elife.45133">10.7554/elife.45133</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31017570">31017570</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6510536">PMC6510536</a></p>
</div>
<div id="ref-mTE5h91z">
<p>48. <strong>Altmetric Scores, Citations, and Publication of Studies Posted as Preprints</strong> <br />
Stylianos Serghiou, John P. A. Ioannidis<br />
<em>JAMA</em> (2018-01-23) <a href="https://doi.org/gftc69">https://doi.org/gftc69</a> <br />
DOI: <a href="https://doi.org/10.1001/jama.2017.21168">10.1001/jama.2017.21168</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29362788">29362788</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5833561">PMC5833561</a></p>
</div>
<div id="ref-oeGSw37I">
<p>49. <strong>A Coefficient of Agreement for Nominal Scales</strong> <br />
Jacob Cohen<br />
<em>Educational and Psychological Measurement</em> (2016-07-02) <a href="https://doi.org/dghsrr">https://doi.org/dghsrr</a> <br />
DOI: <a href="https://doi.org/10.1177/001316446002000104">10.1177/001316446002000104</a></p>
</div>
<div id="ref-TMxQYqVE">
<p>50. <strong>Peer review and the publication process</strong> <br />
Parveen Azam Ali, Roger Watson<br />
<em>Nursing Open</em> (2016-10) <a href="https://doi.org/c4g8">https://doi.org/c4g8</a> <br />
DOI: <a href="https://doi.org/10.1002/nop2.51">10.1002/nop2.51</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27708830">27708830</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5050543">PMC5050543</a></p>
</div>
<div id="ref-1FHYUCO1v">
<p>51. <strong>Biologists debate how to license preprints</strong> <br />
Lindsay McKenzie<br />
<em>Nature</em> (2017-06-16) <a href="https://doi.org/b9fb">https://doi.org/b9fb</a> <br />
DOI: <a href="https://doi.org/10.1038/nature.2017.22161">10.1038/nature.2017.22161</a></p>
</div>
<div id="ref-11GpiE0CN">
<p>52. <strong>The licensing of <em>bioRxiv</em> preprints</strong> <br />
Daniel Himmelstein<br />
<em>Satoshi Village</em> (2016-12-05) <a href="https://blog.dhimmel.com/biorxiv-licenses/">https://blog.dhimmel.com/biorxiv-licenses/</a></p>
</div>
<div id="ref-ddvF5sXo">
<p>53. <strong>Conditional Robust Calibration (CRC): a new computational Bayesian methodology for model parameters estimation and identifiability analysis</strong> <br />
Fortunato Bianconi, Chiara Antonini, Lorenzo Tomassoni, Paolo Valigi<br />
<em>bioRxiv</em> (2017-10-02) <a href="https://doi.org/gg9393">https://doi.org/gg9393</a> <br />
DOI: <a href="https://doi.org/10.1101/197400">10.1101/197400</a></p>
</div>
<div id="ref-18dTP7Wfc">
<p>54. <strong>Machine learning of stochastic gene network phenotypes</strong> <br />
Kyemyung Park, Thorsten Prüstel, Yong Lu, John S. Tsang<br />
<em>Cold Spring Harbor Laboratory</em> (2019-10-31) <a href="https://doi.org/gg94bm">https://doi.org/gg94bm</a> <br />
DOI: <a href="https://doi.org/10.1101/825943">10.1101/825943</a></p>
</div>
<div id="ref-UmFNlo8W">
<p>55. <strong>Notions of similarity for computational biology models</strong> <br />
Ron Henkel, Robert Hoehndorf, Tim Kacprowski, Christian Knüpfer, Wolfram Liebermeister, Dagmar Waltemath<br />
<em>Cold Spring Harbor Laboratory</em> (2016-03-21) <a href="https://doi.org/gg939z">https://doi.org/gg939z</a> <br />
DOI: <a href="https://doi.org/10.1101/044818">10.1101/044818</a></p>
</div>
<div id="ref-16tWubqOj">
<p>56. <strong>GpABC: a Julia package for approximate Bayesian computation with Gaussian process emulation</strong> <br />
Evgeny Tankhilevich, Jonathan Ish-Horowicz, Tara Hameed, Elisabeth Roesch, Istvan Kleijn, Michael PH Stumpf, Fei He<br />
<em>Cold Spring Harbor Laboratory</em> (2019-09-18) <a href="https://doi.org/gg94bj">https://doi.org/gg94bj</a> <br />
DOI: <a href="https://doi.org/10.1101/769299">10.1101/769299</a></p>
</div>
<div id="ref-6gfAR2hM">
<p>57. <strong>SBpipe: a collection of pipelines for automating repetitive simulation and analysis tasks</strong> <br />
Piero Dalle Pezze, Nicolas Le Novère<br />
<em>Cold Spring Harbor Laboratory</em> (2017-02-09) <a href="https://doi.org/gg9392">https://doi.org/gg9392</a> <br />
DOI: <a href="https://doi.org/10.1101/107250">10.1101/107250</a></p>
</div>
<div id="ref-2V2BJL87">
<p>58. <strong>Spatiotemporal proteomics uncovers cathepsin-dependent host cell death during bacterial infection</strong> <br />
Joel Selkrig, Nan Li, Jacob Bobonis, Annika Hausmann, Anna Sueki, Haruna Imamura, Bachir El Debs, Gianluca Sigismondo, Bogdan I. Florea, Herman S. Overkleeft, … Athanasios Typas<br />
<em>bioRxiv</em> (2018-11-07) <a href="https://doi.org/gg94bc">https://doi.org/gg94bc</a> <br />
DOI: <a href="https://doi.org/10.1101/455048">10.1101/455048</a></p>
</div>
<div id="ref-4BUSt1jK">
<p>59. <strong>Systems analysis by mass cytometry identifies susceptibility of latent HIV-infected T cells to targeting of p38 and mTOR pathways</strong> <br />
Linda E. Fong, Victor L. Bass, Serena Spudich, Kathryn Miller-Jensen<br />
<em>Cold Spring Harbor Laboratory</em> (2018-07-19) <a href="https://doi.org/gg9398">https://doi.org/gg9398</a> <br />
DOI: <a href="https://doi.org/10.1101/371922">10.1101/371922</a></p>
</div>
<div id="ref-nzDb3Lvn">
<p>60. <strong>NADPH consumption by L-cystine reduction creates a metabolic vulnerability upon glucose deprivation</strong> <br />
James H. Joly, Alireza Delfarah, Philip S. Phung, Sydney Parrish, Nicholas A. Graham<br />
<em>bioRxiv</em> (2019-08-13) <a href="https://doi.org/gg94bf">https://doi.org/gg94bf</a> <br />
DOI: <a href="https://doi.org/10.1101/733162">10.1101/733162</a></p>
</div>
<div id="ref-1CM9H3mFb">
<p>61. <strong>Inhibition of Bruton’s tyrosine kinase reduces NF-kB and NLRP3 inflammasome activity preventing insulin resistance and microvascular disease</strong> <br />
Gareth S. D. Purvis, Massimo Collino, Haidee M. A. Tavio, Fausto Chiazza, Caroline E. O’Riodan, Lynda Zeboudj, Nick Guisot, Peter Bunyard, David R. Greaves, Christoph Thiemermann<br />
<em>bioRxiv</em> (2019-08-28) <a href="https://doi.org/gg94bg">https://doi.org/gg94bg</a> <br />
DOI: <a href="https://doi.org/10.1101/745943">10.1101/745943</a></p>
</div>
<div id="ref-jmFU2eta">
<p>62. <strong>AKT but not MYC promotes reactive oxygen species-mediated cell death in oxidative culture</strong> <br />
Dongqing Zheng, Jonathan H. Sussman, Matthew P. Jeon, Sydney T. Parrish, Alireza Delfarah, Nicholas A. Graham<br />
<em>bioRxiv</em> (2019-09-01) <a href="https://doi.org/gg94bh">https://doi.org/gg94bh</a> <br />
DOI: <a href="https://doi.org/10.1101/754572">10.1101/754572</a></p>
</div>
<div id="ref-xq8RfP8M">
<p>63. <strong>Pangenome Analysis of Enterobacteria Reveals Richness of Secondary Metabolite Gene Clusters and their Associated Gene Sets</strong> <br />
Omkar S. Mohite, Colton J. Lloyd, Jonathan M. Monk, Tilmann Weber, Bernhard O. Palsson<br />
<em>bioRxiv</em> (2019-09-25) <a href="https://doi.org/gg94bk">https://doi.org/gg94bk</a> <br />
DOI: <a href="https://doi.org/10.1101/781328">10.1101/781328</a></p>
</div>
<div id="ref-18IbfyQMk">
<p>64. <strong>QTG-Finder: a machine-learning based algorithm to prioritize causal genes of quantitative trait loci</strong> <br />
Fan Lin, Jue Fan, Seung Y. Rhee<br />
<em>bioRxiv</em> (2019-04-29) <a href="https://doi.org/gg94bd">https://doi.org/gg94bd</a> <br />
DOI: <a href="https://doi.org/10.1101/484204">10.1101/484204</a></p>
</div>
<div id="ref-TcV6ekeM">
<p>65. <strong>Identification of candidate genes underlying nodulation-specific phenotypes in <em>Medicago truncotula</em> through integration of genome-wide association studies and co-expression networks</strong> <br />
Jean-Michel Michno, Liana T. Burghardt, Junqi Liu, Joseph R. Jeffers, Peter Tiffin, Robert M. Stupar, Chad L. Myers<br />
<em>Cold Spring Harbor Laboratory</em> (2018-08-16) <a href="https://doi.org/gg94bb">https://doi.org/gg94bb</a> <br />
DOI: <a href="https://doi.org/10.1101/392779">10.1101/392779</a></p>
</div>
<div id="ref-1GhuF05Hx">
<p>66. <strong>Raw sequence to target gene prediction: An integrated inference pipeline for ChIP-seq and RNA-seq datasets</strong> <br />
Nisar Wani, Khalid Raza<br />
<em>Cold Spring Harbor Laboratory</em> (2017-11-16) <a href="https://doi.org/gg9394">https://doi.org/gg9394</a> <br />
DOI: <a href="https://doi.org/10.1101/220152">10.1101/220152</a></p>
</div>
<div id="ref-111HFfchz">
<p>67. <strong>The y-ome defines the thirty-four percent of <em>Escherichia coli</em> genes that lack experimental evidence of function</strong> <br />
Sankha Ghatak, Zachary A. King, Anand Sastry, Bernhard O. Palsson<br />
<em>bioRxiv</em> (2018-12-03) <a href="https://doi.org/gg9396">https://doi.org/gg9396</a> <br />
DOI: <a href="https://doi.org/10.1101/328591">10.1101/328591</a></p>
</div>
<div id="ref-17Byktk2J">
<p>68. <strong>The effects of time-varying temperature on delays in genetic networks</strong> <br />
Marcella M Gomez, Richard M Murray, Matthew R Bennett<br />
<em>Cold Spring Harbor Laboratory</em> (2015-09-24) <a href="https://doi.org/gg939x">https://doi.org/gg939x</a> <br />
DOI: <a href="https://doi.org/10.1101/019687">10.1101/019687</a></p>
</div>
<div id="ref-INo8d1Br">
<p>69. <strong>An analog to digital converter creates nuclear localization pulses in yeast calcium signaling</strong> <br />
Ian S Hsu, Bob Strome, Sergey Plotnikov, Alan M Moses<br />
<em>Cold Spring Harbor Laboratory</em> (2018-06-28) <a href="https://doi.org/gg9397">https://doi.org/gg9397</a> <br />
DOI: <a href="https://doi.org/10.1101/357939">10.1101/357939</a></p>
</div>
<div id="ref-wkaMFZ7G">
<p>70. <strong>Nicotinic modulation of hierarchal inhibitory control over prefrontal cortex resting state dynamics: modeling of genetic modification and schizophreniarelated pathology</strong> <br />
Marie Rooy, Fani Koukouli, Uwe Maskos, Boris Gutkin<br />
<em>Cold Spring Harbor Laboratory</em> (2018-04-13) <a href="https://doi.org/gg9395">https://doi.org/gg9395</a> <br />
DOI: <a href="https://doi.org/10.1101/301051">10.1101/301051</a></p>
</div>
<div id="ref-17z6IVXTN">
<p>71. <strong>Electrical propagation of vasodilatory signals in capillary networks</strong> <br />
Pilhwa Lee<br />
<em>Cold Spring Harbor Laboratory</em> (2019-11-13) <a href="https://doi.org/gg94bn">https://doi.org/gg94bn</a> <br />
DOI: <a href="https://doi.org/10.1101/840280">10.1101/840280</a></p>
</div>
<div id="ref-WOFoFT9Q">
<p>72. <strong>Dendritic spine geometry and spine apparatus organization govern the spatiotemporal dynamics of calcium</strong> <br />
Miriam Bell, Tom Bartol, Terrence Sejnowski, Padmini Rangamani<br />
<em>Cold Spring Harbor Laboratory</em> (2019-05-29) <a href="https://doi.org/gg9399">https://doi.org/gg9399</a> <br />
DOI: <a href="https://doi.org/10.1101/386367">10.1101/386367</a></p>
</div>
</div>
<h2 id="supplemental-figures">Supplemental Figures</h2>
<div id="fig:biorxiv_categories" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/e61b10ad5fc0d23db4c6599b460ddd510ae433a5/biorxiv/exploratory_data_analysis/output/figures/preprint_category.png" data-tag="S1" alt="" /><figcaption><span>Figure S1:</span> Neuroscience and bioinformatics are the two most common author-selected topics for bioRxiv preprints.</figcaption>
</figure>
</div>
<div id="fig:biorxiv_v_reference" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/2a121a2a7db820467c6c5c15fffd697b234b5a50/biorxiv/corpora_comparison/output/figures/biorxiv_vs_reference.png" data-tag="S2" alt="" /><figcaption><span>Figure S2:</span> Topic associated tokens are highly enriched when comparing bioRxiv to the New York Times.
The plot on the left (A) is a point range plot of the odds ratio with respect to bioRxiv.
Values greater than one indicate a high association with bioRxiv whereas values less than one indicate high association with the New York Times.
The dotted line provides a breaking point between both categories.
The plot on the right (B) is a bar chart of token frequency appearing in bioRxiv and New York Times respectively.</figcaption>
</figure>
</div>
<div id="fig:pmc_v_reference" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/2a121a2a7db820467c6c5c15fffd697b234b5a50/biorxiv/corpora_comparison/output/figures/pmc_vs_reference.png" data-tag="S3" alt="" /><figcaption><span>Figure S3:</span> Typesetting symbols and biologically relevant tokens are highly enriched when comparing PubMed Central (PMC) to the New York Times.
The plot on the left (A) is a point range plot of the odds ratio with respect to PMC.
Values greater than one indicate a high association with PMC whereas values less than one indicate high association with the New York Times.
The dotted line provides a breaking point between both categories.
The plot on the right (B) is a bar chart of token frequency appearing in PMC and New York Times respectively.</figcaption>
</figure>
</div>
<div id="fig:knn_auc" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/greenelab/annorxiver/472d9b2abd42aecff339fe21c8378f9a556b80d8/pmc/journal_recommendation/output/figures/knn_result.svg" data-tag="S4" style="width:100.0%" alt="" /><figcaption><span>Figure S4:</span> Both classifiers outperform the randomized baseline when predicting a paper’s journal endpoint.
This bargraph shows each model’s accuracy in respect to predicting the training and test set.</figcaption>
</figure>
</div>
<h2 id="supplemental-tables">Supplemental Tables</h2>
<div id="tbl:five_pc1_table" class="tablenos">
<table>
<caption><span>Table 2:</span> Top and bottom five systems biology preprints projected onto the PC1 direction. These preprints contain quantitative and molecular biology concepts respectively. </caption>
<colgroup>
<col style="width: 37%" />
<col style="width: 22%" />
<col style="width: 13%" />
<col style="width: 11%" />
<col style="width: 15%" />
</colgroup>
<thead>
<tr class="header">
<th>Title [citation]</th>
<th>PC_1</th>
<th>License</th>
<th>Figure Thumbnail</th>
<th>Figure Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Conditional Robust Calibration (CRC): a new computational Bayesian methodology for model parameters estimation and identifiability analysis <span class="citation" data-cites="ddvF5sXo">[<a href="#ref-ddvF5sXo" role="doc-biblioref">53</a>]</span></td>
<td>4.700554908074704</td>
<td>None</td>
<td><img src="images/paper-thumbnails/pc1/197400_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2017/10/02/197400/F1.large.jpg</td>
</tr>
<tr class="even">
<td>Machine learning of stochastic gene network phenotypes <span class="citation" data-cites="18dTP7Wfc">[<a href="#ref-18dTP7Wfc" role="doc-biblioref">54</a>]</span></td>
<td>4.410660604449826</td>
<td>CC-BY-NC-ND</td>
<td><img src="images/paper-thumbnails/pc1/825943_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2019/10/31/825943/F5.large.jpg</td>
</tr>
<tr class="odd">
<td>Notions of similarity for computational biology models <span class="citation" data-cites="UmFNlo8W">[<a href="#ref-UmFNlo8W" role="doc-biblioref">55</a>]</span></td>
<td>4.355295926618207</td>
<td>CC-BY-NC-ND</td>
<td><img src="images/paper-thumbnails/pc1/044818_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2016/03/21/044818/F1.large.jpg</td>
</tr>
<tr class="even">
<td>GpABC: a Julia package for approximate Bayesian computation with Gaussian process emulation <span class="citation" data-cites="16tWubqOj">[<a href="#ref-16tWubqOj" role="doc-biblioref">56</a>]</span></td>
<td>4.351517618262304</td>
<td>CC-BY-NC-ND</td>
<td><img src="images/paper-thumbnails/pc1/769299_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2019/09/18/769299/F1.large.jpg</td>
</tr>
<tr class="odd">
<td>SBpipe: a collection of pipelines for automating repetitive simulation and analysis tasks <span class="citation" data-cites="6gfAR2hM">[<a href="#ref-6gfAR2hM" role="doc-biblioref">57</a>]</span></td>
<td>4.321847854182741</td>
<td>CC-BY-NC-ND</td>
<td><img src="images/paper-thumbnails/pc1/107250_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2017/02/09/107250/F1.large.jpg</td>
</tr>
<tr class="even">
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>Spatiotemporal proteomics uncovers cathepsin-dependent host cell death during bacterial infection <span class="citation" data-cites="2V2BJL87">[<a href="#ref-2V2BJL87" role="doc-biblioref">58</a>]</span></td>
<td>-4.263964235099807</td>
<td>CC-BY-ND</td>
<td><img src="images/paper-thumbnails/pc1/455048_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2018/11/07/455048/F1.large.jpg</td>
</tr>
<tr class="even">
<td>Systems analysis by mass cytometry identifies susceptibility of latent HIV-infected T cells to targeting of p38 and mTOR pathways <span class="citation" data-cites="4BUSt1jK">[<a href="#ref-4BUSt1jK" role="doc-biblioref">59</a>]</span></td>
<td>-4.279016673409032</td>
<td>CC-BY-NC-ND</td>
<td><img src="images/paper-thumbnails/pc1/371922_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2018/07/19/371922/F1.large.jpg</td>
</tr>
<tr class="odd">
<td>NADPH consumption by L-cystine reduction creates a metabolic vulnerability upon glucose deprivation <span class="citation" data-cites="nzDb3Lvn">[<a href="#ref-nzDb3Lvn" role="doc-biblioref">60</a>]</span></td>
<td>-4.592209988884236</td>
<td>None</td>
<td><img src="images/paper-thumbnails/pc1/733162_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2019/08/13/733162/F1.large.jpg</td>
</tr>
<tr class="even">
<td>Inhibition of Bruton’s tyrosine kinase reduces NF-kB and NLRP3 inflammasome activity preventing insulin resistance and microvascular disease <span class="citation" data-cites="1CM9H3mFb">[<a href="#ref-1CM9H3mFb" role="doc-biblioref">61</a>]</span></td>
<td>-4.736613689905791</td>
<td>None</td>
<td><img src="images/paper-thumbnails/pc1/745943_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2019/08/28/745943/F1.large.jpg</td>
</tr>
<tr class="odd">
<td>AKT but not MYC promotes reactive oxygen species-mediated cell death in oxidative culture <span class="citation" data-cites="jmFU2eta">[<a href="#ref-jmFU2eta" role="doc-biblioref">62</a>]</span></td>
<td>-4.826793742506695</td>
<td>None</td>
<td><img src="images/paper-thumbnails/pc1/754572_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2019/09/01/754572/F1.large.jpg</td>
</tr>
</tbody>
</table>
</div>
<div id="tbl:five_pc2_table" class="tablenos">
<table>
<caption><span>Table 3:</span> Top and bottom five systems biology preprints projected onto the PC2 direction. These preprints contain bioinformatis and neuroscience concepts respectively. </caption>
<colgroup>
<col style="width: 37%" />
<col style="width: 16%" />
<col style="width: 22%" />
<col style="width: 11%" />
<col style="width: 11%" />
</colgroup>
<thead>
<tr class="header">
<th>Title [citation]</th>
<th>PC_2</th>
<th>License</th>
<th>Figure Thumbnail</th>
<th>Figure Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Pangenome Analysis of Enterobacteria Reveals Richness of Secondary Metabolite Gene Clusters and their Associated Gene Sets <span class="citation" data-cites="xq8RfP8M">[<a href="#ref-xq8RfP8M" role="doc-biblioref">63</a>]</span></td>
<td>3.5865702659438883</td>
<td>CC-BY-ND</td>
<td><img src="images/paper-thumbnails/pc2/781328_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2019/09/25/781328/F1.large.jpg</td>
</tr>
<tr class="even">
<td>QTG-Finder: a machine-learning based algorithm to prioritize causal genes of quantitative trait loci <span class="citation" data-cites="18IbfyQMk">[<a href="#ref-18IbfyQMk" role="doc-biblioref">64</a>]</span></td>
<td>3.470388383023157</td>
<td>None</td>
<td><img src="images/paper-thumbnails/pc2/484204_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2019/04/29/484204/F1.large.jpg</td>
</tr>
<tr class="odd">
<td>Identification of candidate genes underlying nodulation-specific phenotypes in Medicago truncotula through integration of genome-wide association studies and co-expression networks <span class="citation" data-cites="TcV6ekeM">[<a href="#ref-TcV6ekeM" role="doc-biblioref">65</a>]</span></td>
<td>3.3814906334073953</td>
<td>CC-BY-NC-ND</td>
<td><img src="images/paper-thumbnails/pc2/392779_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2018/08/16/392779/F1.large.jpg</td>
</tr>
<tr class="even">
<td>Raw sequence to target gene prediction: An integrated inference pipeline for ChIP-seq and RNA-seq datasets <span class="citation" data-cites="1GhuF05Hx">[<a href="#ref-1GhuF05Hx" role="doc-biblioref">66</a>]</span></td>
<td>3.3632576028389742</td>
<td>None</td>
<td><img src="images/paper-thumbnails/pc2/220152_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2017/11/16/220152/F3.large.jpg</td>
</tr>
<tr class="odd">
<td>The y-ome defines the thirty-four percent of Escherichia coli genes that lack experimental evidence of function <span class="citation" data-cites="111HFfchz">[<a href="#ref-111HFfchz" role="doc-biblioref">67</a>]</span></td>
<td>3.28742786641417</td>
<td>CC-BY</td>
<td><img src="images/paper-thumbnails/pc2/328591_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2018/12/03/328591/F1.large.jpg</td>
</tr>
<tr class="even">
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>The effects of time-varying temperature on delays in genetic networks <span class="citation" data-cites="17Byktk2J">[<a href="#ref-17Byktk2J" role="doc-biblioref">68</a>]</span></td>
<td>-2.7047102478958056</td>
<td>None</td>
<td><img src="images/paper-thumbnails/pc2/019687_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2015/09/24/019687/F1.large.jpg</td>
</tr>
<tr class="even">
<td>An analog to digital converter creates nuclear localization pulses in yeast calcium signaling <span class="citation" data-cites="INo8d1Br">[<a href="#ref-INo8d1Br" role="doc-biblioref">69</a>]</span></td>
<td>-2.775745000260895</td>
<td>None</td>
<td><img src="images/paper-thumbnails/pc2/357939_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2018/06/28/357939/F1.large.jpg</td>
</tr>
<tr class="odd">
<td>Nicotinic modulation of hierarchal inhibitory control over prefrontal cortex resting state dynamics: modeling of genetic modification and schizophreniarelated pathology <span class="citation" data-cites="wkaMFZ7G">[<a href="#ref-wkaMFZ7G" role="doc-biblioref">70</a>]</span></td>
<td>-3.047342382798414</td>
<td>None</td>
<td><img src="images/paper-thumbnails/pc2/301051_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2018/04/13/301051/F1.large.jpg</td>
</tr>
<tr class="even">
<td>Electrical propagation of vasodilatory signals in capillary networks <span class="citation" data-cites="17z6IVXTN">[<a href="#ref-17z6IVXTN" role="doc-biblioref">71</a>]</span></td>
<td>-3.107715578793087</td>
<td>CC-BY-NC-ND</td>
<td><img src="images/paper-thumbnails/pc2/840280_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2019/11/13/840280/F1.large.jpg</td>
</tr>
<tr class="odd">
<td>Dendritic spine geometry and spine apparatus organization govern the spatiotemporal dynamics of calcium <span class="citation" data-cites="WOFoFT9Q">[<a href="#ref-WOFoFT9Q" role="doc-biblioref">72</a>]</span></td>
<td>-3.21533499072831</td>
<td>CC-BY-NC-ND</td>
<td><img src="images/paper-thumbnails/pc2/386367_thumbnail.png" title="fig:" /></td>
<td>https://www.biorxiv.org/content/biorxiv/early/2019/05/29/386367/F1.large.jpg</td>
</tr>
</tbody>
</table>
</div>
<!-- default theme -->

<style>
    /* import google fonts */
    @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
    @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

    /* -------------------------------------------------- */
    /* global */
    /* -------------------------------------------------- */

    /* all elements */
    * {
        /* force sans-serif font unless specified otherwise */
        font-family: "Open Sans", "Helvetica", sans-serif;

        /* prevent text inflation on some mobile browsers */
        -webkit-text-size-adjust: none !important;
        -moz-text-size-adjust: none !important;
        -o-text-size-adjust: none !important;
        text-size-adjust: none !important;
    }

    @media only screen {
        /* "page" element */
        body {
            position: relative;
            box-sizing: border-box;
            font-size: 12pt;
            line-height: 1.5;
            max-width: 8.5in;
            margin: 20px auto;
            padding: 40px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* "page" element */
        body {
            padding: 20px;
            margin: 0;
            border-radius: 0;
            border: none;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
            background: none;
        }
    }

    /* -------------------------------------------------- */
    /* headings */
    /* -------------------------------------------------- */

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
        margin: 20px 0;
        padding: 0;
        font-weight: bold;
    }

    /* biggest heading */
    h1 {
        margin: 40px 0;
        text-align: center;
    }

    /* second biggest heading */
    h2 {
        margin-top: 30px;
        padding-bottom: 5px;
        border-bottom: solid 1px #bdbdbd;
    }

    /* heading font sizes */
    h1 {
        font-size: 2em;
    }
    h2 {
        font-size: 1.5em;
    }
    h3{
        font-size: 1.35em;
    }
    h4 {
        font-size: 1.25em;
    }
    h5 {
        font-size: 1.15em;
    }
    h6 {
        font-size: 1em;
    }

    /* -------------------------------------------------- */
    /* manuscript header */
    /* -------------------------------------------------- */

    /* manuscript title */
    header > h1 {
        margin: 0;
    }

    /* manuscript title caption text (ie "automatically generated on") */
    header + p {
        text-align: center;
        margin-top: 10px;
    }

    /* -------------------------------------------------- */
    /* text elements */
    /* -------------------------------------------------- */

    /* links */
    a {
        color: #2196f3;
        overflow-wrap: break-word;
    }

    /* normal links (not empty, not button link, not syntax highlighting link) */
    a:not(:empty):not(.button):not(.sourceLine) {
        padding-left: 1px;
        padding-right: 1px;
    }

    /* superscripts and subscripts */
    sub,
    sup {
        /* prevent from affecting line height */
        line-height: 0;
    }

    /* unordered and ordered lists*/
    ul,
    ol {
        padding-left: 20px;
    }

    /* class for styling text semibold */
    .semibold {
        font-weight: 600;
    }

    /* class for styling elements horizontally left aligned */
    .left {
        display: block;
        text-align: left;
        margin-left: auto;
        margin-right: 0;
        justify-content: left;
    }

    /* class for styling elements horizontally centered */
    .center {
        display: block;
        text-align: center;
        margin-left: auto;
        margin-right: auto;
        justify-content: center;
    }

    /* class for styling elements horizontally right aligned */
    .right {
        display: block;
        text-align: right;
        margin-left: 0;
        margin-right: auto;
        justify-content: right;
    }

    /* -------------------------------------------------- */
    /* section elements */
    /* -------------------------------------------------- */

    /* horizontal divider line */
    hr {
        border: none;
        height: 1px;
        background: #bdbdbd;
    }

    /* paragraphs, horizontal dividers, figures, tables, code */
    p,
    hr,
    figure,
    table,
    pre {
        /* treat all as "paragraphs", with consistent vertical margins */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* figures */
    /* -------------------------------------------------- */

    /* figure */
    figure {
        max-width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure caption */
    figcaption {
        padding: 0;
        padding-top: 10px;
    }

    /* figure image element */
    figure > img,
    figure > svg {
        max-width: 100%;
        display: block;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure auto-number */
    img + figcaption > span:first-of-type,
    svg + figcaption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* tables */
    /* -------------------------------------------------- */

    /* table */
    table {
        border-collapse: collapse;
        border-spacing: 0;
        width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* table cells */
    th,
    td {
        border: solid 1px #bdbdbd;
        padding: 10px;
        /* squash table if too wide for page by forcing line breaks */
        overflow-wrap: break-word;
        word-break: break-word;
    }

    /* header row and even rows */
    th,
    tr:nth-child(2n) {
        background-color: #fafafa;
    }

    /* odd rows */
    tr:nth-child(2n + 1) {
        background-color: #ffffff;
    }

    /* table caption */
    caption {
        text-align: left;
        padding: 0;
        padding-bottom: 10px;
    }

    /* table auto-number */
    table > caption > span:first-of-type,
    div.table_wrapper > table > caption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* code */
    /* -------------------------------------------------- */

    /* multi-line code block */
    pre {
        padding: 10px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
        break-inside: avoid;
        text-align: left;
    }

    /* inline code, ie code within normal text */
    :not(pre) > code {
        padding: 0 4px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
    }

    /* code text */
    /* apply all children, to reach syntax highlighting sub-elements */
    code,
    code * {
        /* force monospace font */
        font-family: "Source Code Pro", "Courier New", monospace;
    }

    /* -------------------------------------------------- */
    /* quotes */
    /* -------------------------------------------------- */

    /* quoted text */
    blockquote {
        margin: 0;
        padding: 0;
        border-left: 4px solid #bdbdbd;
        padding-left: 16px;
        break-inside: avoid;
    }

    /* -------------------------------------------------- */
    /* banners */
    /* -------------------------------------------------- */

    /* info banners */
    .banner {
        box-sizing: border-box;
        display: block;
        position: relative;
        width: 100%;
        margin-top: 20px;
        margin-bottom: 20px;
        padding: 20px;
        text-align: center;
    }

    /* paragraph in banner */
    .banner > p {
        margin: 0;
    }

    /* -------------------------------------------------- */
    /* highlight colors */
    /* -------------------------------------------------- */

    .white {
        background: #ffffff;
    }
    .lightgrey {
        background: #eeeeee;
    }
    .grey {
        background: #757575;
    }
    .darkgrey {
        background: #424242;
    }
    .black {
        background: #000000;
    }
    .lightred {
        background: #ffcdd2;
    }
    .lightyellow {
        background: #ffecb3;
    }
    .lightgreen {
        background: #dcedc8;
    }
    .lightblue {
        background: #e3f2fd;
    }
    .lightpurple {
        background: #f3e5f5;
    }
    .red {
        background: #f44336;
    }
    .orange {
        background: #ff9800;
    }
    .yellow {
        background: #ffeb3b;
    }
    .green {
        background: #4caf50;
    }
    .blue {
        background: #2196f3;
    }
    .purple {
        background: #9c27b0;
    }
    .white,
    .lightgrey,
    .lightred,
    .lightyellow,
    .lightgreen,
    .lightblue,
    .lightpurple,
    .orange,
    .yellow,
    .white a,
    .lightgrey a,
    .lightred a,
    .lightyellow a,
    .lightgreen a,
    .lightblue a,
    .lightpurple a,
    .orange a,
    .yellow a {
        color: #000000;
    }
    .grey,
    .darkgrey,
    .black,
    .red,
    .green,
    .blue,
    .purple,
    .grey a,
    .darkgrey a,
    .black a,
    .red a,
    .green a,
    .blue a,
    .purple a {
        color: #ffffff;
    }

    /* -------------------------------------------------- */
    /* buttons */
    /* -------------------------------------------------- */

    /* class for styling links like buttons */
    .button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        margin: 5px;
        padding: 10px 20px;
        font-size: 0.75em;
        font-weight: 600;
        text-transform: uppercase;
        text-decoration: none;
        letter-spacing: 1px;
        background: none;
        color: #2196f3;
        border: solid 1px #bdbdbd;
        border-radius: 5px;
    }

    /* buttons when hovered */
    .button:hover:not([disabled]),
    .icon_button:hover:not([disabled]) {
        cursor: pointer;
        background: #f5f5f5;
    }

    /* buttons when disabled */
    .button[disabled],
    .icon_button[disabled] {
        opacity: 0.35;
        pointer-events: none;
    }

    /* class for styling buttons containg only single icon */
    .icon_button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        text-decoration: none;
        margin: 0;
        padding: 0;
        background: none;
        border-radius: 5px;
        border: none;
        width: 20px;
        height: 20px;
        min-width: 20px;
        min-height: 20px;
    }

    /* icon button inner svg image */
    .icon_button > svg {
        height: 16px;
    }

    /* -------------------------------------------------- */
    /* icons */
    /* -------------------------------------------------- */

    /* class for styling icons inline with text */
    .inline_icon {
        height: 1em;
        position: relative;
        top: 0.125em;
    }

    /* -------------------------------------------------- */
    /* print control */
    /* -------------------------------------------------- */

    @media print {
        @page {
            /* suggested printing margin */
            margin: 0.5in;
        }

        /* document and "page" elements */
        html, body {
            margin: 0;
            padding: 0;
            width: 100%;
            height: 100%;
        }

        /* "page" element */
        body {
            font-size: 11pt !important;
            line-height: 1.35;
        }

        /* all headings */
        h1,
        h2,
        h3,
        h4,
        h5,
        h6 {
            margin: 15px 0;
        }

        /* figures and tables */
        figure, table {
            font-size: 0.85em;
        }

        /* table cells */
        th,
        td {
            padding: 5px;
        }

        /* shrink font awesome icons */
        i.fas,
        i.fab,
        i.far,
        i.fal {
            transform: scale(0.85);
        }

        /* decrease banner margins */
        .banner {
            margin-top: 15px;
            margin-bottom: 15px;
            padding: 15px;
        }

        /* class for centering an element vertically on its own page */
        .page_center {
            margin: auto;
            width: 100%;
            height: 100%;
            display: flex;
            align-items: center;
            vertical-align: middle;
            break-before: page;
            break-after: page;
        }

        /* always insert a page break before the element */
        .page_break_before {
            break-before: page;
        }

        /* always insert a page break after the element */
        .page_break_after {
            break-after: page;
        }

        /* avoid page break before the element */
        .page_break_before_avoid {
            break-before: avoid;
        }

        /* avoid page break after the element */
        .page_break_after_avoid {
            break-after: avoid;
        }

        /* avoid page break inside the element */
        .page_break_inside_avoid {
            break-inside: avoid;
        }
    }

    /* -------------------------------------------------- */
    /* override pandoc css quirks */
    /* -------------------------------------------------- */

    .sourceCode {
        /* prevent unsightly overflow in wide code blocks */
        overflow: auto !important;
    }

    div.sourceCode {
        /* prevent background fill on top-most code block  container */
        background: none !important;
    }

    .sourceCode * {
        /* force consistent line spacing */
        line-height: 1.5 !important;
    }

    div.sourceCode {
        /* style code block margins same as <pre> element */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* tablenos */
    /* -------------------------------------------------- */

    /* tablenos wrapper */
    .tablenos {
        /* show scrollbar on tables if necessary to prevent overflow */
        width: 100%;
        margin: 20px 0;
    }

    .tablenos > table {
        /* move margins from table to table_wrapper to allow margin collapsing */
        margin: 0;
    }

    @media only screen {
        /* tablenos wrapper */
        .tablenos {
            /* show scrollbar on tables if necessary to prevent overflow */
            overflow-x: auto !important;
        }

        .tablenos th,
        .tablenos td {
            overflow-wrap: unset !important;
            word-break: unset !important;
        }

        /* table in wrapper */
        .tablenos table,
        .tablenos table * {
            /* don't break table words */
            overflow-wrap: normal !important;
        }
    }

    /* -------------------------------------------------- */
    /* mathjax */
    /* -------------------------------------------------- */

    /* mathjax containers */
    .math.display > span:not(.MathJax_Preview) {
        /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
        display: flex !important;
        overflow-x: auto !important;
        overflow-y: hidden !important;
        justify-content: center;
        align-items: center;
        margin: 0 !important;
    }

    /* right click menu */
    .MathJax_Menu {
        border-radius: 5px !important;
        border: solid 1px #bdbdbd !important;
        box-shadow: none !important;
    }

    /* equation auto-number */
    span[id^="eq:"] > span.math.display + span {
        font-weight: 600;
    }

    /* equation */
    span[id^="eq:"] > span.math.display > span {
        /* nudge to make room for equation auto-number and anchor */
        margin-right: 60px !important;
    }

    /* -------------------------------------------------- */
    /* anchors plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anchor button */
        .anchor {
            opacity: 0;
            margin-left: 5px;
        }

        /* anchor buttons within <h2>'s */
        h2 .anchor {
            margin-left: 10px;
        }

        /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
        *:hover > .anchor,
        .anchor:hover,
        .anchor:focus {
            opacity: 1;
        }

        /* anchor button when hovered */
        .anchor:hover {
            cursor: pointer;
        }
    }

    /* always show anchor button on devices with no mouse/hover ability */
    @media (hover: none) {
        .anchor {
            opacity: 1;
        }
    }

    /* always hide anchor button on print */
    @media only print {
        .anchor {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* accordion plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* accordion arrow button */
        .accordion_arrow {
            margin-right: 10px;
        }

        /* arrow icon when <h2> data-collapsed attribute true */
        h2[data-collapsed="true"] > .accordion_arrow > svg {
            transform: rotate(-90deg);
        }

        /* all elements (except <h2>'s) when data-collapsed attribute true */
        *:not(h2)[data-collapsed="true"] {
            display: none;
        }

        /* accordion arrow button when hovered and <h2>'s when hovered */
        .accordion_arrow:hover,
        h2[data-collapsed="true"]:hover,
        h2[data-collapsed="false"]:hover {
            cursor: pointer;
        }
    }

    /* always hide accordion arrow button on print */
    @media only print {
        .accordion_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* tooltips plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* tooltip container */
        #tooltip {
            position: absolute;
            width: 50%;
            min-width: 240px;
            max-width: 75%;
            z-index: 1;
        }

        /* tooltip content */
        #tooltip_content {
            margin-bottom: 5px;
            padding: 20px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
            overflow-wrap: break-word;
        }

        /* tooltip copy of paragraphs and figures */
        #tooltip_content > p,
        #tooltip_content > figure {
            margin: 0;
            max-height: 320px;
            overflow-y: auto;
        }

        /* tooltip copy of <img> */
        #tooltip_content > figure > img,
        #tooltip_content > figure > svg {
            max-height: 260px;
        }

        /* navigation bar */
        #tooltip_nav_bar {
            margin-top: 10px;
            text-align: center;
        }

        /* navigation bar previous/next buton */
        #tooltip_nav_bar > .icon_button {
            position: relative;
            top: 3px;
        }

        /* navigation bar previous button */
        #tooltip_nav_bar > .icon_button:first-of-type {
            margin-right: 5px;
        }

        /* navigation bar next button */
        #tooltip_nav_bar > .icon_button:last-of-type {
            margin-left: 5px;
        }
    }

    /* always hide tooltip on print */
    @media only print {
        #tooltip {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* jump to first plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* jump button */
        .jump_arrow {
            position: relative;
            top: 0.125em;
            margin-right: 5px;
        }
    }

    /* always hide jump button on print */
    @media only print {
        .jump_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* link highlight plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anything with data-highlighted attribute true */
        [data-highlighted="true"] {
            background: #ffeb3b;
        }

        /* anything with data-selected attribute true */
        [data-selected="true"] {
            background: #ff8a65 !important;
        }

        /* animation definition for glow */
        @keyframes highlight_glow {
            0% {
                background: none;
            }
            10% {
                background: #bbdefb;
            }
            100% {
                background: none;
            }
        }

        /* anything with data-glow attribute true */
        [data-glow="true"] {
            animation: highlight_glow 2s;
        }
    }

    /* -------------------------------------------------- */
    /* table of contents plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* toc panel */
        #toc_panel {
            box-sizing: border-box;
            position: fixed;
            top: 0;
            left: 0;
            background: #ffffff;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            z-index: 2;
        }

        /* toc panel when closed */
        #toc_panel[data-open="false"] {
            min-width: 60px;
            width: 60px;
            height: 60px;
            border-right: solid 1px #bdbdbd;
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc panel when open */
        #toc_panel[data-open="true"] {
            min-width: 260px;
            max-width: 480px;
            /* keep panel edge consistent distance away from "page" edge */
            width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
            bottom: 0;
            border-right: solid 1px #bdbdbd;
        }

        /* toc panel header */
        #toc_header {
            box-sizing: border-box;
            display: flex;
            flex-direction: row;
            align-items: center;
            height: 60px;
            margin: 0;
            padding: 20px;
        }

        /* toc panel header when hovered */
        #toc_header:hover {
            cursor: pointer;
        }

        /* toc panel header when panel open */
        #toc_panel[data-open="true"] > #toc_header {
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc open/close header button */
        #toc_button {
            margin-right: 20px;
        }

        /* hide toc list and header text when closed */
        #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
        #toc_panel[data-open="false"] > #toc_list {
            display: none;
        }

        /* toc list of entries */
        #toc_list {
            box-sizing: border-box;
            width: 100%;
            padding: 20px;
            position: absolute;
            top: calc(60px + 1px);
            bottom: 0;
            overflow: auto;
        }

        /* toc entry, link to section in document */
        .toc_link {
            display: block;
            padding: 5px;
            position: relative;
            font-weight: 600;
            text-decoration: none;
        }

        /* toc entry when hovered or when "viewed" */
        .toc_link:hover,
        .toc_link[data-viewing="true"] {
            background: #f5f5f5;
        }

        /* toc entry, level 1 indentation */
        .toc_link[data-level="1"] {
            margin-left: 0;
        }

        /* toc entry, level 2 indentation */
        .toc_link[data-level="2"] {
            margin-left: 20px;
        }

        /* toc entry, level 3 indentation */
        .toc_link[data-level="3"] {
            margin-left: 40px;
        }

        /* toc entry, level 4 indentation */
        .toc_link[data-level="4"] {
            margin-left: 60px;
        }

        /* toc entry bullets */
        #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
            position: absolute;
            left: -15px;
            top: -1px;
            font-size: 1.5em;
        }

        /* toc entry, level 2 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
            content: "\2022";
        }

        /* toc entry, level 3 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
            content: "\25AB";
        }

        /* toc entry, level 4 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
            content: "-";
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* push <body> ("page") element down to make room for toc icon */
        .toc_body_nudge {
            padding-top: 60px;
        }

        /* toc icon when panel closed and not hovered */
        #toc_panel[data-open="false"]:not(:hover) {
            background: rgba(255, 255, 255, 0.75);
        }
    }

    /* always hide toc panel on print */
    @media only print {
        #toc_panel {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* lightbox plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* regular <img> in document when hovered */
        img.lightbox_document_img:hover {
            cursor: pointer;
        }

        .body_no_scroll {
            overflow: hidden !important;
        }

        /* screen overlay */
        #lightbox_overlay {
            display: flex;
            flex-direction: column;
            position: fixed;
            left: 0;
            top: 0;
            right: 0;
            bottom: 0;
            background: rgba(0, 0, 0, 0.75);
            z-index: 3;
        }

        /* middle area containing lightbox image */
        #lightbox_image_container {
            flex-grow: 1;
            display: flex;
            justify-content: center;
            align-items: center;
            overflow: hidden;
            position: relative;
            padding: 20px;
        }

        /* bottom area containing caption */
        #lightbox_bottom_container {
            display: flex;
            justify-content: center;
            align-items: center;
            height: 100px;
            min-height: 100px;
            max-height: 100px;
            background: rgba(0, 0, 0, 0.5);
        }

        /* image number info text box */
        #lightbox_number_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            left: 2px;
            top: 0;
            z-index: 4;
        }

        /* zoom info text box */
        #lightbox_zoom_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            right: 2px;
            top: 0;
            z-index: 4;
        }

        /* copy of image caption */
        #lightbox_caption {
            box-sizing: border-box;
            display: inline-block;
            width: 100%;
            max-height: 100%;
            padding: 10px 0;
            text-align: center;
            overflow-y: auto;
            color: #ffffff;
        }

        /* navigation previous/next button */
        .lightbox_button {
            width: 100px;
            height: 100%;
            min-width: 100px;
            min-height: 100%;
            color: #ffffff;
        }

        /* navigation previous/next button when hovered */
        .lightbox_button:hover {
            background: none !important;
        }

        /* navigation button icon */
        .lightbox_button > svg {
            height: 25px;
        }

        /* figure auto-number */
        #lightbox_caption > span:first-of-type {
            font-weight: bold;
            margin-right: 5px;
        }

        /* lightbox image when hovered */
        #lightbox_img:hover {
            cursor: grab;
        }

        /* lightbox image when grabbed */
        #lightbox_img:active {
            cursor: grabbing;
        }
    }

    /* when on screen < 480px wide */
    @media only screen and (max-width: 480px) {
        /* make navigation buttons skinnier on small screens to make more room for caption text */
        .lightbox_button {
            width: 50px;
            min-width: 50px;
        }
    }

    /* always hide lightbox on print */
    @media only print {
        #lightbox_overlay {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* hypothesis (annotations) plugin */
    /* -------------------------------------------------- */

    /* hypothesis activation button */
    #hypothesis_button {
        box-sizing: border-box;
        position: fixed;
        top: 0;
        right: 0;
        width: 60px;
        height: 60px;
        background: #ffffff;
        border-radius: 0;
        border-left: solid 1px #bdbdbd;
        border-bottom: solid 1px #bdbdbd;
        box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
        z-index: 2;
    }

    /* hypothesis button svg */
    #hypothesis_button > svg {
        position: relative;
        top: -4px;
    }

    /* hypothesis annotation count */
    #hypothesis_count {
        position: absolute;
        left: 0;
        right: 0;
        bottom: 5px;
    }

    /* side panel */
    .annotator-frame {
        width: 280px !important;
    }

    /* match highlight color to rest of theme */
    .annotator-highlights-always-on .annotator-hl {
        background-color: #ffeb3b !important;
    }

    /* match focused color to rest of theme */
    .annotator-hl.annotator-hl-focused {
        background-color: #ff8a65 !important;
    }

    /* match bucket bar color to rest of theme */
    .annotator-bucket-bar {
        background: #f5f5f5 !important;
    }

    /* always hide button, toolbar, and tooltip on print */
    @media only print {
        #hypothesis_button {
            display: none;
        }

        .annotator-frame {
            display: none !important;
        }

        hypothesis-adder {
            display: none !important;
        }
    }
</style>
<!-- anchors plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds an anchor next to each of a certain type
        // of element that provides a human-readable url to that specific
        // item/position in the document (eg "manuscript.html#abstract"). It
        // also makes it such that scrolling out of view of a target removes
        // its identifier from the url.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'anchors';

        // default plugin options
        const options = {
            // which types of elements to add anchors next to, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3, [id^="fig:"], [id^="tbl:"], [id^="eq:"]',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // add anchor to each element of specified types
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements)
                addAnchor(element);

            // attach scroll listener to window
            window.addEventListener('scroll', onScroll);
        }

        // when window is scrolled
        function onScroll() {
            // if url has hash and user has scrolled out of view of hash
            // target, remove hash from url
            const tolerance = 100;
            const target = getHashTarget();
            if (target) {
                if (
                    target.getBoundingClientRect().top >
                        window.innerHeight + tolerance ||
                    target.getBoundingClientRect().bottom < 0 - tolerance
                )
                    history.pushState(null, null, ' ');
            }
        }

        // add anchor to element
        function addAnchor(element) {
            let addTo; // element to add anchor button to

            // if figure or table, modify withId and addTo to get expected
            // elements
            if (element.id.indexOf('fig:') === 0) {
                addTo = element.querySelector('figcaption');
            } else if (element.id.indexOf('tbl:') === 0) {
                addTo = element.querySelector('caption');
            } else if (element.id.indexOf('eq:') === 0) {
                addTo = element.querySelector('.eqnos-number');
            }

            addTo = addTo || element;
            const id = element.id || null;

            // do not add anchor if element doesn't have assigned id.
            // id is generated by pandoc and is assumed to be unique and
            // human-readable
            if (!id)
                return;

            // create anchor button
            const anchor = document.createElement('a');
            anchor.innerHTML = document.querySelector('.icon_link').innerHTML;
            anchor.title = 'Link to this part of the document';
            anchor.classList.add('icon_button', 'anchor');
            anchor.dataset.ignore = 'true';
            anchor.href = '#' + id;
            addTo.appendChild(anchor);
        }

        // get element that is target of link or url hash
        function getHashTarget() {
            const hash = window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');
            if (id.indexOf('tbl:') === 0)
                target = target.querySelector('table');

            return target;
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- link icon -->

<template class="icon_link">
    <!-- modified from: https://fontawesome.com/icons/link -->
    <svg width="16" height="16" viewBox="0 0 512 512">
        <path
            fill="currentColor"
            d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
        ></path>
    </svg>
</template>
<!-- accordion plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows sections of content under <h2> headings
        // to be collapsible.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'accordion';

        // default plugin options
        const options = {
            // whether to always start expanded ('false'), always start
            // collapsed ('true'), or start collapsed when screen small ('auto')
            startCollapsed: 'auto',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <h2> heading
            const headings = document.querySelectorAll('h2');
            for (const heading of headings) {
                addArrow(heading);

                // start expanded/collapsed based on option
                if (
                    options.startCollapsed === 'true' ||
                    (options.startCollapsed === 'auto' && isSmallScreen())
                )
                    collapseHeading(heading);
                else
                    expandHeading(heading);
            }

            // attach hash change listener to window
            window.addEventListener('hashchange', onHashChange);
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                goToElement(target);
        }

        // add arrow to heading
        function addArrow(heading) {
            // add arrow button
            const arrow = document.createElement('button');
            arrow.innerHTML = document.querySelector(
                '.icon_angle_down'
            ).innerHTML;
            arrow.classList.add('icon_button', 'accordion_arrow');
            heading.insertBefore(arrow, heading.firstChild);

            // attach click listener to heading and button
            heading.addEventListener('click', onHeadingClick);
            arrow.addEventListener('click', onArrowClick);
        }

        // determine if on mobile-like device with small screen
        function isSmallScreen() {
            return Math.min(window.innerWidth, window.innerHeight) < 480;
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get element that is target of hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');
            if (id.indexOf('tbl:') === 0)
                target = target.querySelector('table');

            return target;
        }

        // when <h2> heading is clicked
        function onHeadingClick(event) {
            // only collapse if <h2> itself is target of click (eg, user did
            // not click on anchor within <h2>)
            if (event.target === this)
                toggleCollapse(this);
        }

        // when arrow button is clicked
        function onArrowClick() {
            toggleCollapse(this.parentNode);
        }

        // collapse section if expanded, expand if collapsed
        function toggleCollapse(heading) {
            if (heading.dataset.collapsed === 'false')
                collapseHeading(heading);
            else
                expandHeading(heading);
        }

        // elements to exclude from collapse, such as table of contents panel,
        // hypothesis panel, etc
        const exclude = '#toc_panel, div.annotator-frame, #lightbox_overlay';

        // collapse section
        function collapseHeading(heading) {
            heading.setAttribute('data-collapsed', 'true');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'true');
        }

        // expand section
        function expandHeading(heading) {
            heading.setAttribute('data-collapsed', 'false');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'false');
        }

        // get list of elements between this <h2> and next <h2> or <h1>
        // ("children" of the <h2> section)
        function getChildren(heading) {
            return nextUntil(heading, 'h2, h1', exclude);
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get list of elements after a start element up to element matching
        // query
        function nextUntil(element, query, exclude) {
            const elements = [];
            while (element = element.nextElementSibling, element) {
                if (element.matches(query))
                    break;
                if (!element.matches(exclude))
                    elements.push(element);
            }
            return elements;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
    <!-- modified from: https://fontawesome.com/icons/angle-down -->
    <svg width="16" height="16" viewBox="0 0 448 512">
        <path
            fill="currentColor"
            d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
        ></path>
    </svg>
</template>
<!-- tooltips plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when the user hovers or
        // focuses a link to a citation or figure, a tooltip appears with a
        // preview of the reference content, along with arrows to navigate
        // between instances of the same reference in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tooltips';

        // default plugin options
        const options = {
            // whether user must click off to close tooltip instead of just
            // un-hovering
            clickClose: 'false',
            // delay (in ms) between opening and closing tooltip
            delay: '100',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach hover and focus listeners to link
                link.addEventListener('mouseover', onLinkHover);
                link.addEventListener('mouseleave', onLinkUnhover);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('touchend', onLinkTouch);
            }

            // attach mouse, key, and resize listeners to window
            window.addEventListener('mousedown', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('keyup', onKeyUp);
            window.addEventListener('resize', onResize);
        }

        // when link is hovered
        function onLinkHover() {
            // function to open tooltip
            const delayOpenTooltip = function() {
                openTooltip(this);
            }.bind(this);

            // run open function after delay
            this.openTooltipTimer = window.setTimeout(
                delayOpenTooltip,
                options.delay
            );
        }

        // when mouse leaves link
        function onLinkUnhover() {
            // cancel opening tooltip
            window.clearTimeout(this.openTooltipTimer);

            // don't close on unhover if option specifies
            if (options.clickClose === 'true')
                return;

            // function to close tooltip
            const delayCloseTooltip = function() {
                // if tooltip open and if mouse isn't over tooltip, close
                const tooltip = document.getElementById('tooltip');
                if (tooltip && !tooltip.matches(':hover'))
                    closeTooltip();
            };

            // run close function after delay
            this.closeTooltipTimer = window.setTimeout(
                delayCloseTooltip,
                options.delay
            );
        }

        // when link is focused (tabbed to)
        function onLinkFocus(event) {
            openTooltip(this);
        }

        // when link is touched on touch screen
        function onLinkTouch(event) {
            // attempt to force hover state on first tap always, and trigger
            // regular link click (and navigation) on second tap
            if (event.target === document.activeElement)
                event.target.click();
            else {
                document.activeElement.blur();
                event.target.focus();
            }
            if (event.cancelable)
                event.preventDefault();
            event.stopPropagation();
            return false;
        }

        // when mouse is clicked anywhere in window
        function onClick(event) {
            closeTooltip();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'tooltip_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'tooltip_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeTooltip();
                    break;
            }
        }

        // when window is resized or zoomed
        function onResize() {
            closeTooltip();
        }

        // get all links of types we wish to handle
        function getLinks() {
            const queries = [];
            // exclude buttons, anchor links, toc links, etc
            const exclude =
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            queries.push('a[href^="#ref-"]' + exclude); // citation links
            queries.push('a[href^="#fig:"]' + exclude); // figure links
            const query = queries.join(', ');
            return document.querySelectorAll(query);
        }

        // get links with same target, get index of link in set, get total
        // same links
        function getSameLinks(link) {
            const sameLinks = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    sameLinks.push(otherLink);
            }

            return {
                elements: sameLinks,
                index: sameLinks.indexOf(link),
                total: sameLinks.length
            };
        }

        // open tooltip
        function openTooltip(link) {
            // delete tooltip if it exists, start fresh
            closeTooltip();

            // make tooltip element
            const tooltip = makeTooltip(link);

            // if source couldn't be found and tooltip not made, exit
            if (!tooltip)
                return;

            // make navbar elements
            const navBar = makeNavBar(link);
            if (navBar)
                tooltip.firstElementChild.appendChild(navBar);

            // attach tooltip to page
            document.body.appendChild(tooltip);

            // position tooltip
            const position = function() {
                positionTooltip(link);
            };
            position();

            // if tooltip contains images, position again after they've loaded
            const imgs = tooltip.querySelectorAll('img');
            for (const img of imgs)
                img.addEventListener('load', position);
        }

        // close (delete) tooltip
        function closeTooltip() {
            const tooltip = document.getElementById('tooltip');
            if (tooltip)
                tooltip.remove();
        }

        // make tooltip
        function makeTooltip(link) {
            // get target element that link points to
            const source = getSource(link);

            // if source can't be found, exit
            if (!source)
                return;

            // create new tooltip
            const tooltip = document.createElement('div');
            tooltip.id = 'tooltip';
            const tooltipContent = document.createElement('div');
            tooltipContent.id = 'tooltip_content';
            tooltip.appendChild(tooltipContent);

            // make copy of source node and put in tooltip
            const sourceCopy = makeCopy(source);
            tooltipContent.appendChild(sourceCopy);

            // attach mouse event listeners
            tooltip.addEventListener('click', onTooltipClick);
            tooltip.addEventListener('mousedown', onTooltipClick);
            tooltip.addEventListener('touchstart', onTooltipClick);
            tooltip.addEventListener('mouseleave', onTooltipUnhover);

            // (for interaction with lightbox plugin)
            // transfer click on tooltip copied img to original img
            const sourceImg = source.querySelector('img');
            const sourceCopyImg = sourceCopy.querySelector('img');
            if (sourceImg && sourceCopyImg) {
                const clickImg = function() {
                    sourceImg.click();
                    closeTooltip();
                };
                sourceCopyImg.addEventListener('click', clickImg);
            }

            return tooltip;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // when tooltip is clicked
        function onTooltipClick(event) {
            // when user clicks on tooltip, stop click from transferring
            // outside of tooltip (eg, click off to close tooltip, or eg click
            // off to unhighlight same refs)
            event.stopPropagation();
        }

        // when tooltip is unhovered
        function onTooltipUnhover(event) {
            if (options.clickClose === 'true')
                return;

            // make sure new mouse/touch/focus no longer over tooltip or any
            // element within it
            const tooltip = document.getElementById('tooltip');
            if (!tooltip)
                return;
            if (this.contains(event.relatedTarget))
                return;

            closeTooltip();
        }

        // make nav bar to go betwen prev/next instances of same reference
        function makeNavBar(link) {
            // find other links to the same source
            const sameLinks = getSameLinks(link);

            // don't show nav bar when singular reference
            if (sameLinks.total <= 1)
                return;

            // find prev/next links with same target
            const prevLink = getPrevLink(link, sameLinks);
            const nextLink = getNextLink(link, sameLinks);

            // create nav bar
            const navBar = document.createElement('div');
            navBar.id = 'tooltip_nav_bar';
            const text = sameLinks.index + 1 + ' of ' + sameLinks.total;

            // create nav bar prev/next buttons
            const prevButton = document.createElement('button');
            const nextButton = document.createElement('button');
            prevButton.id = 'tooltip_prev_button';
            nextButton.id = 'tooltip_next_button';
            prevButton.title =
                'Jump to the previous occurence of this item in the document [←]';
            nextButton.title =
                'Jump to the next occurence of this item in the document [→]';
            prevButton.classList.add('icon_button');
            nextButton.classList.add('icon_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;
            navBar.appendChild(prevButton);
            navBar.appendChild(document.createTextNode(text));
            navBar.appendChild(nextButton);

            // attach click listeners to buttons
            prevButton.addEventListener('click', function() {
                onPrevNextClick(link, prevLink);
            });
            nextButton.addEventListener('click', function() {
                onPrevNextClick(link, nextLink);
            });

            return navBar;
        }

        // get previous link with same target
        function getPrevLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if < 1
            let index;
            if (sameLinks.index - 1 >= 0)
                index = sameLinks.index - 1;
            else
                index = sameLinks.total - 1;
            return sameLinks.elements[index];
        }

        // get next link with same target
        function getNextLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if > total
            let index;
            if (sameLinks.index + 1 <= sameLinks.total - 1)
                index = sameLinks.index + 1;
            else
                index = 0;
            return sameLinks.elements[index];
        }

        // get element that is target of link or url hash
        function getSource(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if ref or figure, modify target to get expected element
            if (id.indexOf('ref-') === 0)
                target = target.querySelector('p');
            else if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');

            return target;
        }

        // when prev/next arrow button is clicked
        function onPrevNextClick(link, prevNextLink) {
            if (link && prevNextLink)
                goToElement(prevNextLink, window.innerHeight * 0.5);
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // determine position to place tooltip based on link position in
        // viewport and tooltip size
        function positionTooltip(link, left, top) {
            const tooltipElement = document.getElementById('tooltip');
            if (!tooltipElement)
                return;

            // get convenient vars for position/dimensions of
            // link/tooltip/page/view
            link = getRectInPage(link);
            const tooltip = getRectInPage(tooltipElement);
            const view = getRectInPage();

            // horizontal positioning
            if (left)
                // use explicit value
                left = left;
            else if (link.left + tooltip.width < view.right)
                // fit tooltip to right of link
                left = link.left;
            else if (link.right - tooltip.width > view.left)
                // fit tooltip to left of link
                left = link.right - tooltip.width;
            // center tooltip in view
            else
                left = (view.right - view.left) / 2 - tooltip.width / 2;

            // vertical positioning
            if (top)
                // use explicit value
                top = top;
            else if (link.top - tooltip.height > view.top)
                // fit tooltip above link
                top = link.top - tooltip.height;
            else if (link.bottom + tooltip.height < view.bottom)
                // fit tooltip below link
                top = link.bottom;
            else {
                // center tooltip in view
                top = view.top + view.height / 2 - tooltip.height / 2;
                // nudge off of link to left/right if possible
                if (link.right + tooltip.width < view.right)
                    left = link.right;
                else if (link.left - tooltip.width > view.left)
                    left = link.left - tooltip.width;
            }

            tooltipElement.style.left = left + 'px';
            tooltipElement.style.top = top + 'px';
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get position of element relative to page
        function getRectInPage(element) {
            const rect = getRectInView(element);
            const body = getRectInView(document.body);

            const newRect = {};
            newRect.left = rect.left - body.left;
            newRect.top = rect.top - body.top;
            newRect.right = rect.right - body.left;
            newRect.bottom = rect.bottom - body.top;
            newRect.width = rect.width;
            newRect.height = rect.height;

            return newRect;
        }

        // (for interaction with accordion plugin)
        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // (for interaction with accordion plugin)
        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- jump to first plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds a button next to each reference entry,
        // figure, and table that jumps the page to the first occurrence of a
        // link to that item in the manuscript.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'jumpToFirst';

        // default plugin options
        const options = {
            // whether to add buttons next to reference entries
            references: 'true',
            // whether to add buttons next to figures
            figures: 'true',
            // whether to add buttons next to tables
            tables: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            if (options.references !== 'false')
                makeReferenceButtons();
            if (options.figures !== 'false')
                makeFigureButtons();
            if (options.tables !== 'false')
                makeTableButtons();
        }

        // when jump button clicked
        function onButtonClick() {
            const first = getFirstOccurrence(this.dataset.id);
            if (!first)
                return;

            // update url hash so navigating "back" in history will return
            // user to jump button
            window.location.hash = this.dataset.id;
            // scroll to link
            window.setTimeout(function() {
                goToElement(first, window.innerHeight * 0.5);
            }, 0);
        }

        // get first occurence of link to item in document
        function getFirstOccurrence(id) {
            let query = 'a';
            query += '[href="#' + id + '"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelector(query);
        }

        // add button next to each reference entry
        function makeReferenceButtons() {
            const references = document.querySelectorAll('div[id^="ref-"]');
            for (const reference of references) {
                // get reference id and element to add button to
                const id = reference.id;
                const container = reference.firstElementChild;
                const first = getFirstOccurrence(id);

                // if can't find link to reference, ignore
                if (!first)
                    continue;

                // make jump button
                let button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this reference in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.innerHTML = button.outerHTML + container.innerHTML;
                button = container.firstElementChild;
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeFigureButtons() {
            const figures = document.querySelectorAll('[id^="fig:"]');
            for (const figure of figures) {
                // get figure id and element to add button to
                const id = figure.id;
                const container = figure.querySelector('figcaption') || figure;
                const first = getFirstOccurrence(id);

                // if can't find link to figure, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this figure in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeTableButtons() {
            const tables = document.querySelectorAll('[id^="tbl:"]');
            for (const table of tables) {
                // get ref id and element to add button to
                const id = table.id;
                const container = table.querySelector('caption') || table;
                const first = getFirstOccurrence(id);

                // if can't find link to table, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this table in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
    <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
    <svg width="16" height="16" viewBox="0 0 320 512">
        <path
            fill="currentColor"
            d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
        ></path>
    </svg>
</template>
<!-- link highlight plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user hovers or
        // focuses a link, other links that have the same target will be
        // highlighted. It also makes it such that when clicking a link, the
        // target of the link (eg reference, figure, table) is briefly
        // highlighted.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'linkHighlight';

        // default plugin options
        const options = {
            // whether to also highlight links that go to external urls
            externalLinks: 'false',
            // whether user must click off to unhighlight instead of just
            // un-hovering
            clickUnhighlight: 'false',
            // whether to also highlight links that are unique
            highlightUnique: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach mouse and focus listeners to link
                link.addEventListener('mouseenter', onLinkFocus);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('mouseleave', onLinkUnhover);
            }

            // attach click and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('hashchange', onHashChange);

            // run hash change on window load in case user has navigated
            // directly to hash
            onHashChange();
        }

        // when link is focused (tabbed to) or hovered
        function onLinkFocus() {
            highlight(this);
        }

        // when link is unhovered
        function onLinkUnhover() {
            if (options.clickUnhighlight !== 'true')
                unhighlightAll();
        }

        // when the mouse is clicked anywhere in window
        function onClick(event) {
            unhighlightAll();
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                glowElement(target);
        }

        // get element that is target of link or url hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            return target;
        }

        // start glow sequence on an element
        function glowElement(element) {
            const startGlow = function() {
                onGlowEnd();
                element.dataset.glow = 'true';
                element.addEventListener('animationend', onGlowEnd);
            };
            const onGlowEnd = function() {
                element.removeAttribute('data-glow');
                element.removeEventListener('animationend', onGlowEnd);
            };
            startGlow();
        }

        // highlight link and all others with same target
        function highlight(link) {
            // force unhighlight all to start fresh
            unhighlightAll();

            // get links with same target
            if (!link)
                return;
            const sameLinks = getSameLinks(link);

            // if link unique and option is off, exit and don't highlight
            if (sameLinks.length <= 1 && options.highlightUnique !== 'true')
                return;

            // highlight all same links, and "select" (special highlight) this
            // one
            for (const sameLink of sameLinks) {
                if (sameLink === link)
                    sameLink.setAttribute('data-selected', 'true');
                else
                    sameLink.setAttribute('data-highlighted', 'true');
            }
        }

        // unhighlight all links
        function unhighlightAll() {
            const links = getLinks();
            for (const link of links) {
                link.setAttribute('data-selected', 'false');
                link.setAttribute('data-highlighted', 'false');
            }
        }

        // get links with same target
        function getSameLinks(link) {
            const results = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    results.push(otherLink);
            }
            return results;
        }

        // get all links of types we wish to handle
        function getLinks() {
            let query = 'a';
            if (options.externalLinks !== 'true')
                query += '[href^="#"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelectorAll(query);
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin provides a "table of contents" (toc) panel on
        // the side of the document that allows the user to conveniently
        // navigate between sections of the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tableOfContents';

        // default plugin options
        const options = {
            // which types of elements to add links for, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3',
            // whether toc starts open. use 'true' or 'false', or 'auto' to
            // use 'true' behavior when screen wide enough and 'false' when not
            startOpen: 'false',
            // whether toc closes when clicking on toc link. use 'true' or
            // 'false', or 'auto' to use 'false' behavior when screen wide
            // enough and 'true' when not
            clickClose: 'auto',
            // if list item is more than this many characters, text will be
            // truncated
            charLimit: '50',
            // whether or not to show bullets next to each toc item
            bullets: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // make toc panel and populate with entries (links to document
            // sections)
            const panel = makePanel();
            if (!panel)
                return;
            makeEntries(panel);
            // attach panel to document after making entries, so 'toc' heading
            // in panel isn't included in toc
            document.body.insertBefore(panel, document.body.firstChild);

            // initial panel state
            if (
                options.startOpen === 'true' ||
                (options.startOpen === 'auto' && !isSmallScreen())
            )
                openPanel();
            else
                closePanel();

            // attach click, scroll, and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('scroll', onScroll);
            window.addEventListener('hashchange', onScroll);
            window.addEventListener('keyup', onKeyUp);
            onScroll();

            // add class to push document body down out of way of toc button
            document.body.classList.add('toc_body_nudge');
        }

        // determine if screen wide enough to fit toc panel
        function isSmallScreen() {
            // in default theme:
            // 816px = 8.5in = width of "page" (<body>) element
            // 260px = min width of toc panel (*2 for both sides of <body>)
            return window.innerWidth < 816 + 260 * 2;
        }

        // when mouse is clicked anywhere in window
        function onClick() {
            if (isSmallScreen())
                closePanel();
        }

        // when window is scrolled or hash changed
        function onScroll() {
            highlightViewed();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            // close on esc
            if (event.key === 'Escape')
                closePanel();
        }

        // find entry of currently viewed document section in toc and highlight
        function highlightViewed() {
            const firstId = getFirstInView(options.typesQuery);

            // get toc entries (links), unhighlight all, then highlight viewed
            const list = document.getElementById('toc_list');
            if (!firstId || !list)
                return;
            const links = list.querySelectorAll('a');
            for (const link of links)
                link.dataset.viewing = 'false';
            const link = list.querySelector('a[href="#' + firstId + '"]');
            if (!link)
                return;
            link.dataset.viewing = 'true';
        }

        // get first or previous toc listed element in top half of view
        function getFirstInView(query) {
            // get all elements matching query and with id
            const elements = document.querySelectorAll(query);
            const elementsWithIds = [];
            for (const element of elements) {
                if (element.id)
                    elementsWithIds.push(element);
            }


            // get first or previous element in top half of view
            for (let i = 0; i < elementsWithIds.length; i++) {
                const element = elementsWithIds[i];
                const prevElement = elementsWithIds[Math.max(0, i - 1)];
                if (element.getBoundingClientRect().top >= 0) {
                    if (
                        element.getBoundingClientRect().top <
                        window.innerHeight / 2
                    )
                        return element.id;
                    else
                        return prevElement.id;
                }
            }
        }

        // make panel
        function makePanel() {
            // create panel
            const panel = document.createElement('div');
            panel.id = 'toc_panel';
            if (options.bullets === 'true')
                panel.dataset.bullets = 'true';

            // create header
            const header = document.createElement('div');
            header.id = 'toc_header';

            // create toc button
            const button = document.createElement('button');
            button.id = 'toc_button';
            button.innerHTML = document.querySelector('.icon_th_list').innerHTML;
            button.title = 'Table of Contents';
            button.classList.add('icon_button');

            // create header text
            const text = document.createElement('h4');
            text.innerHTML = 'Table of Contents';

            // create container for toc list
            const list = document.createElement('div');
            list.id = 'toc_list';

            // attach click listeners
            panel.addEventListener('click', onPanelClick);
            header.addEventListener('click', onHeaderClick);
            button.addEventListener('click', onButtonClick);

            // attach elements
            header.appendChild(button);
            header.appendChild(text);
            panel.appendChild(header);
            panel.appendChild(list);

            return panel;
        }

        // create toc entries (links) to each element of the specified types
        function makeEntries(panel) {
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements) {
                // do not add link if element doesn't have assigned id
                if (!element.id)
                    continue;

                // create link/list item
                const link = document.createElement('a');
                link.classList.add('toc_link');
                switch (element.tagName.toLowerCase()) {
                    case 'h1':
                        link.dataset.level = '1';
                        break;
                    case 'h2':
                        link.dataset.level = '2';
                        break;
                    case 'h3':
                        link.dataset.level = '3';
                        break;
                    case 'h4':
                        link.dataset.level = '4';
                        break;
                }
                link.title = element.innerText;
                let text = element.innerText;
                if (text.length > options.charLimit)
                    text = text.slice(0, options.charLimit) + '...';
                link.innerHTML = text;
                link.href = '#' + element.id;
                link.addEventListener('click', onLinkClick);

                // attach link
                panel.querySelector('#toc_list').appendChild(link);
            }
        }

        // when panel is clicked
        function onPanelClick(event) {
            // stop click from propagating to window/document and closing panel
            event.stopPropagation();
        }

        // when header itself is clicked
        function onHeaderClick(event) {
            togglePanel();
        }

        // when button is clicked
        function onButtonClick(event) {
            togglePanel();
            // stop header underneath button from also being clicked
            event.stopPropagation();
        }

        // when link is clicked
        function onLinkClick(event) {
            if (
                options.clickClose === 'true' ||
                (options.clickClose === 'auto' && isSmallScreen())
            )
                closePanel();
            else
                openPanel();
        }

        // open panel if closed, close if opened
        function togglePanel() {
            const panel = document.getElementById('toc_panel');
            if (!panel)
                return;

            if (panel.dataset.open === 'true')
                closePanel();
            else
                openPanel();
        }

        // open panel
        function openPanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'true';
        }

        // close panel
        function closePanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'false';
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- th list icon -->

<template class="icon_th_list">
    <!-- modified from: https://fontawesome.com/icons/th-list -->
    <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
        <path
            fill="currentColor"
            d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
            tabindex="-1"
        ></path>
    </svg>
</template>
<!-- lightbox plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user clicks on an
        // image, the image fills the screen and the user can pan/drag/zoom
        // the image and navigate between other images in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'lightbox';

        // default plugin options
        const options = {
            // list of possible zoom/scale factors
            zoomSteps:
                '0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1,' +
                '1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8',
            // whether to fit image to view ('fit'), display at 100% and shrink
            // if necessary ('shrink'), or always display at 100% ('100')
            defaultZoom: 'fit',
            // whether to zoom in/out toward center of view ('true') or mouse
            // ('false')
            centerZoom: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <img> element
            const imgs = document.querySelectorAll('figure > img');
            let count = 1;
            for (const img of imgs) {
                img.classList.add('lightbox_document_img');
                img.dataset.number = count;
                img.dataset.total = imgs.length;
                img.addEventListener('click', openLightbox);
                count++;
            }

            // attach mouse and key listeners to window
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('keyup', onKeyUp);
        }

        // when mouse is moved anywhere in window
        function onWindowMouseMove(event) {
            window.mouseX = event.clientX;
            window.mouseY = event.clientY;
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'lightbox_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'lightbox_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeLightbox();
                    break;
            }
        }

        // open lightbox
        function openLightbox() {
            const lightbox = makeLightbox(this);
            if (!lightbox)
                return;

            blurBody(lightbox);
            document.body.appendChild(lightbox);
        }

        // make lightbox
        function makeLightbox(img) {
            // delete lightbox if it exists, start fresh
            closeLightbox();

            // create screen overlay containing lightbox
            const overlay = document.createElement('div');
            overlay.id = 'lightbox_overlay';

            // create image info boxes
            const numberInfo = document.createElement('div');
            const zoomInfo = document.createElement('div');
            numberInfo.id = 'lightbox_number_info';
            zoomInfo.id = 'lightbox_zoom_info';

            // create container for image
            const imageContainer = document.createElement('div');
            imageContainer.id = 'lightbox_image_container';
            const lightboxImg = makeLightboxImg(
                img,
                imageContainer,
                numberInfo,
                zoomInfo
            );
            imageContainer.appendChild(lightboxImg);

            // create bottom container for caption and navigation buttons
            const bottomContainer = document.createElement('div');
            bottomContainer.id = 'lightbox_bottom_container';
            const caption = makeCaption(img);
            const prevButton = makePrevButton(img);
            const nextButton = makeNextButton(img);
            bottomContainer.appendChild(prevButton);
            bottomContainer.appendChild(caption);
            bottomContainer.appendChild(nextButton);

            // attach top middle and bottom to overlay
            overlay.appendChild(numberInfo);
            overlay.appendChild(zoomInfo);
            overlay.appendChild(imageContainer);
            overlay.appendChild(bottomContainer);

            return overlay;
        }

        // make <img> object that is intuitively draggable and zoomable
        function makeLightboxImg(
            sourceImg,
            container,
            numberInfoBox,
            zoomInfoBox
        ) {
            // create copy of source <img>
            const img = sourceImg.cloneNode(true);
            img.classList.remove('lightbox_document_img');
            img.removeAttribute('id');
            img.removeAttribute('width');
            img.removeAttribute('height');
            img.style.position = 'unset';
            img.style.margin = '0';
            img.style.padding = '0';
            img.style.width = '';
            img.style.height = '';
            img.style.minWidth = '';
            img.style.minHeight = '';
            img.style.maxWidth = '';
            img.style.maxHeight = '';
            img.id = 'lightbox_img';

            // build sorted list of unique zoomSteps, always including a 100%
            let zoomSteps = [];
            const optionsZooms = options.zoomSteps.split(/[^0-9.]/);
            for (const optionZoom of optionsZooms) {
                const newZoom = parseFloat(optionZoom);
                if (newZoom && !zoomSteps.includes(newZoom))
                    zoomSteps.push(newZoom);
            }
            if (!zoomSteps.includes(1))
                zoomSteps.push(1);
            zoomSteps = zoomSteps.sort(function sortNumber(a, b) {
                return a - b;
            });

            // <img> object property variables
            let zoom = 1;
            let translateX = 0;
            let translateY = 0;
            let clickMouseX = undefined;
            let clickMouseY = undefined;
            let clickTranslateX = undefined;
            let clickTranslateY = undefined;

            updateNumberInfo();

            // update image numbers displayed in info box
            function updateNumberInfo() {
                numberInfoBox.innerHTML =
                    sourceImg.dataset.number + ' of ' + sourceImg.dataset.total;
            }

            // update zoom displayed in info box
            function updateZoomInfo() {
                let zoomInfo = zoom * 100;
                if (!Number.isInteger(zoomInfo))
                    zoomInfo = zoomInfo.toFixed(2);
                zoomInfoBox.innerHTML = zoomInfo + '%';
            }

            // move to closest zoom step above current zoom
            const zoomIn = function() {
                for (const zoomStep of zoomSteps) {
                    if (zoomStep > zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                updateTransform();
            };

            // move to closest zoom step above current zoom
            const zoomOut = function() {
                zoomSteps.reverse();
                for (const zoomStep of zoomSteps) {
                    if (zoomStep < zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                zoomSteps.reverse();

                updateTransform();
            };

            // update display of <img> based on scale/translate properties
            const updateTransform = function() {
                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                // get new width/height after scale
                const rect = img.getBoundingClientRect();
                // limit translate
                translateX = Math.max(translateX, -rect.width / 2);
                translateX = Math.min(translateX, rect.width / 2);
                translateY = Math.max(translateY, -rect.height / 2);
                translateY = Math.min(translateY, rect.height / 2);

                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                updateZoomInfo();
            };

            // fit <img> to container
            const fit = function() {
                // no x/y offset, 100% zoom by default
                translateX = 0;
                translateY = 0;
                zoom = 1;

                // widths of <img> and container
                const imgWidth = img.naturalWidth;
                const imgHeight = img.naturalHeight;
                const containerWidth = parseFloat(
                    window.getComputedStyle(container).width
                );
                const containerHeight = parseFloat(
                    window.getComputedStyle(container).height
                );

                // how much zooming is needed to fit <img> to container
                const xRatio = imgWidth / containerWidth;
                const yRatio = imgHeight / containerHeight;
                const maxRatio = Math.max(xRatio, yRatio);
                const newZoom = 1 / maxRatio;

                // fit <img> to container according to option
                if (options.defaultZoom === 'shrink') {
                    if (maxRatio > 1)
                        zoom = newZoom;
                } else if (options.defaultZoom === 'fit')
                    zoom = newZoom;

                updateTransform();
            };

            // when mouse wheel is rolled anywhere in container
            const onContainerWheel = function(event) {
                if (!event)
                    return;

                // let ctrl + mouse wheel to zoom behave as normal
                if (event.ctrlKey)
                    return;

                // prevent normal scroll behavior
                event.preventDefault();
                event.stopPropagation();

                // point around which to scale img
                const viewRect = container.getBoundingClientRect();
                const viewX = (viewRect.left + viewRect.right) / 2;
                const viewY = (viewRect.top + viewRect.bottom) / 2;
                const originX = options.centerZoom === 'true' ? viewX : mouseX;
                const originY = options.centerZoom === 'true' ? viewY : mouseY;

                // get point on image under origin
                const oldRect = img.getBoundingClientRect();
                const oldPercentX = (originX - oldRect.left) / oldRect.width;
                const oldPercentY = (originY - oldRect.top) / oldRect.height;

                // increment/decrement zoom
                if (event.deltaY < 0)
                    zoomIn();
                if (event.deltaY > 0)
                    zoomOut();

                // get offset between previous image point and origin
                const newRect = img.getBoundingClientRect();
                const offsetX =
                    originX - (newRect.left + newRect.width * oldPercentX);
                const offsetY =
                    originY - (newRect.top + newRect.height * oldPercentY);

                // translate image to keep image point under origin
                translateX += offsetX;
                translateY += offsetY;

                // perform translate
                updateTransform();
            };

            // when container is clicked
            function onContainerClick(event) {
                // if container itself is target of click, and not other
                // element above it
                if (event.target === this)
                    closeLightbox();
            }

            // when mouse button is pressed on image
            const onImageMouseDown = function(event) {
                // store original mouse position relative to image
                clickMouseX = window.mouseX;
                clickMouseY = window.mouseY;
                clickTranslateX = translateX;
                clickTranslateY = translateY;
                event.stopPropagation();
                event.preventDefault();
            };

            // when mouse button is released anywhere in window
            const onWindowMouseUp = function(event) {
                // reset original mouse position
                clickMouseX = undefined;
                clickMouseY = undefined;
                clickTranslateX = undefined;
                clickTranslateY = undefined;

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mouseup', onWindowMouseUp);
            };

            // when mouse is moved anywhere in window
            const onWindowMouseMove = function(event) {
                if (
                    clickMouseX === undefined ||
                    clickMouseY === undefined ||
                    clickTranslateX === undefined ||
                    clickTranslateY === undefined
                )
                    return;

                // offset image based on original and current mouse position
                translateX = clickTranslateX + window.mouseX - clickMouseX;
                translateY = clickTranslateY + window.mouseY - clickMouseY;
                updateTransform();
                event.preventDefault();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mousemove', onWindowMouseMove);
            };

            // when window is resized
            const onWindowResize = function(event) {
                fit();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('resize', onWindowResize);
            };

            // attach the necessary event listeners
            img.addEventListener('dblclick', fit);
            img.addEventListener('mousedown', onImageMouseDown);
            container.addEventListener('wheel', onContainerWheel);
            container.addEventListener('mousedown', onContainerClick);
            container.addEventListener('touchstart', onContainerClick);
            window.addEventListener('mouseup', onWindowMouseUp);
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('resize', onWindowResize);

            // run fit() after lightbox atttached to document and <img> Loaded
            // so needed container and img dimensions available
            img.addEventListener('load', fit);

            return img;
        }

        // make caption
        function makeCaption(img) {
            const caption = document.createElement('div');
            caption.id = 'lightbox_caption';
            const captionSource = img.nextElementSibling;
            if (captionSource.tagName.toLowerCase() === 'figcaption') {
                const captionCopy = makeCopy(captionSource);
                caption.innerHTML = captionCopy.innerHTML;
            }

            caption.addEventListener('touchstart', function(event) {
                event.stopPropagation();
            });

            return caption;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // make button to jump to previous image in document
        function makePrevButton(img) {
            const prevButton = document.createElement('button');
            prevButton.id = 'lightbox_prev_button';
            prevButton.title = 'Jump to the previous image in the document [←]';
            prevButton.classList.add('icon_button', 'lightbox_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;

            // attach click listeners to button
            prevButton.addEventListener('click', function() {
                getPrevImg(img).click();
            });

            return prevButton;
        }

        // make button to jump to next image in document
        function makeNextButton(img) {
            const nextButton = document.createElement('button');
            nextButton.id = 'lightbox_next_button';
            nextButton.title = 'Jump to the next image in the document [→]';
            nextButton.classList.add('icon_button', 'lightbox_button');
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;

            // attach click listeners to button
            nextButton.addEventListener('click', function() {
                getNextImg(img).click();
            });

            return nextButton;
        }

        // get previous image in document
        function getPrevImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if < 1
            if (index - 1 >= 0)
                index--;
            else
                index = imgs.length - 1;
            return imgs[index];
        }

        // get next image in document
        function getNextImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if > total
            if (index + 1 <= imgs.length - 1)
                index++;
            else
                index = 0;
            return imgs[index];
        }

        // close lightbox
        function closeLightbox() {
            focusBody();

            const lightbox = document.getElementById('lightbox_overlay');
            if (lightbox)
                lightbox.remove();
        }

        // make all elements behind lightbox non-focusable
        function blurBody(overlay) {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.tabIndex = -1;
            document.body.classList.add('body_no_scroll');
        }

        // make all elements focusable again
        function focusBody() {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.removeAttribute('tabIndex');
            document.body.classList.remove('body_no_scroll');
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- attributes plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows arbitrary HTML attributes to be attached
        // to (almost) any element. Place an HTML comment inside or next to the
        // desired element in the format <!-- $attribute="value" -->

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'attributes';

        // default plugin options
        const options = {
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // get list of comments in document
            const comments = findComments();

            for(const comment of comments)
                if (comment.parentElement)
                    addAttributes(
                        comment.parentElement,
                        comment.nodeValue.trim()
                    );
        }

        // add html attributes to specified element based on string of 
        // html attributes and values
        function addAttributes(element, text) {
            // regex's for finding attribute/value pairs in the format of
            // attribute="value" or attribute='value
            const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
            const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

            // loop through attribute/value pairs
            let match;
            while(match = text.match(regex2) || text.match(regex1)) {
                // get attribute and value from regex capture groups
                let attribute = match[1];
                let value = match[2];

                // remove from string
                text = text.substring(match.index + match[0].length);

                if (!attribute || !value)
                    break;

                // set attribute of parent element
                try {
                    element.setAttribute(attribute, value);
                } catch(error) {
                    console.log(error);
                }

                // special case for colspan
                if (attribute === 'colspan')
                    removeTableCells(element, value);
            }
        }

        // get list of comment elements in document
        function findComments() {
            const comments = [];

            // iterate over comment nodes in document
            function acceptNode(node) {
                return NodeFilter.FILTER_ACCEPT;
            }
            const iterator = document.createNodeIterator(
                document.body,
                NodeFilter.SHOW_COMMENT,
                acceptNode
            );
            let node;
            while(node = iterator.nextNode())
                comments.push(node);

            return comments;
        }

        // remove certain number of cells after specified cell
        function removeTableCells(cell, number) {
            number = parseInt(number);
            if (!number)
                return;

            // remove elements
            for(; number > 1; number--) {
                if (cell.nextElementSibling)
                    cell.nextElementSibling.remove();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<!-- mathjax plugin configuration -->

<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        "CommonHTML": { linebreaks: { automatic: true } },
        "HTML-CSS": { linebreaks: { automatic: true } },
        "SVG": { linebreaks: { automatic: true } },
        "fast-preview": { disabled: true }
  });
</script>

<!-- mathjax plugin -->

<script
    src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
    integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
    crossorigin="anonymous"
>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'MathJax' allows the proper rendering of
    // math/equations written in LaTeX.

    // https://www.mathjax.org/
</script>
<!-- annotations plugin -->

<script>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'Hypothesis' allows public annotation of the
    // manuscript.

    // https://web.hypothes.is/

    // plugin configuration
    window.hypothesisConfig = function() {
        return {
            branding: {
                accentColor: '#2196f3',
                appBackgroundColor: '#f8f8f8',
                ctaBackgroundColor: '#f8f8f8',
                ctaTextColor: '#000000',
                selectionFontFamily: 'Open Sans, Helvetica, sans serif',
                annotationFontFamily: 'Open Sans, Helvetica, sans serif'
            }
        };
    };

    // hypothesis client script
    const embed = 'https://hypothes.is/embed.js';
    // hypothesis annotation count query url
    const query = 'https://api.hypothes.is/api/search?limit=0&url='

    
    // start script
    function start() {
        const button = makeButton();
        document.body.insertBefore(button, document.body.firstChild);
        insertCount(button);
    }

    // make button
    function makeButton() {
        // create button
        const button = document.createElement('button');
        button.id = 'hypothesis_button';
        button.innerHTML = document.querySelector('.icon_hypothesis').innerHTML;
        button.title = 'Hypothesis annotations';
        button.classList.add('icon_button');

        function onClick(event) {
            onButtonClick(event, button);
        }

        // attach click listeners
        button.addEventListener('click', onClick);

        return button;
    }

    // insert annotations count
    async function insertCount(button) {
        // get annotation count from Hypothesis based on url
        let count = '-';
        try {
            const canonical = document.querySelector('link[rel="canonical"]');
            const location = window.location;
            const url = encodeURIComponent((canonical || location).href);
            const response = await fetch(query + url);
            const json = await response.json();
            count = json.total || '-';
        } catch(error) {
            console.log(error);
        }
        
        // put count into button
        const counter = document.createElement('span');
        counter.id = 'hypothesis_count';
        counter.innerHTML = count;
        button.title = 'View ' + count + ' Hypothesis annotations';
        button.append(counter);
    }

    // when button is clicked
    function onButtonClick(event, button) {
        const script = document.createElement('script');
        script.src = embed;
        document.body.append(script);
        button.remove();
    }

    window.addEventListener('load', start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
    <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
    <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
        <path
            fill="currentColor"
            d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
            tabindex="-1"
        ></path>
    </svg>
</template>
<!-- analytics plugin -->

<!-- copy and paste code from Google Analytics or similar service here -->
</body>
</html>
